# German translation for sbackup
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the sbackup package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: sbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-10-03 14:59+0200\n"
"PO-Revision-Date: 2010-09-29 23:08+0000\n"
"Last-Translator: Jean-Peer Lorenz <Unknown>\n"
"Language-Team: German <de@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2010-10-01 09:04+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#: src/sbackup/backupproc.py:168 src/sbackup/pkginfo.py:68
#, python-format
msgid "SBackup Daemon <%(login)s@%(hostname)s>"
msgstr "SBackup Daemon <%(login)s@%(hostname)s>"

#: src/sbackup/backupproc.py:172
#, python-format
msgid "[SBackup] [%(profile)s] Report of %(date)s"
msgstr "[SBackup] [%(profile)s] Bericht vom %(date)s"

#: src/sbackup/backupproc.py:177 src/sbackup/core/profile_handler.py:278
msgid "No log file specified."
msgstr "Es wurde keine Protokolldatei angegeben."

#: src/sbackup/backupproc.py:182
msgid "Unable to find log file."
msgstr "Protokolldatei konnte nicht gefunden werden."

#: src/sbackup/backupproc.py:248
msgid ""
"Critical Error: No configuration file for the default profile was found!\n"
"\n"
"Now continue processing remaining profiles."
msgstr ""
"Kritischer Fehler: Es wurde keine Konfiguration für das Standardprofil "
"gefunden!\n"
"\n"
"Es wird mit den verbleibenden Profilen fort gefahren."

#: src/sbackup/backupproc.py:297
msgid "Backup was canceled by user."
msgstr "Die Sicherung wurde vom Benutzer abgebrochen."

#: src/sbackup/backupproc.py:302
msgid "An error occurred during the cancellation of backup:"
msgstr "Während des Abbruchs der Sicherung ist ein Fehler aufgetreten:"

#: src/sbackup/backupproc.py:308
msgid "An error occurred during the backup:"
msgstr "Während der Sicherung trat folgender Fehler auf:"

#: src/sbackup/backupproc.py:325
msgid "Error when sending email:"
msgstr "Fehler beim Versenden der E-Mail"

#: src/sbackup/backupproc.py:356
msgid "The following error(s) occurred before:"
msgstr "Die folgenden Fehler sind bisher aufgetreten:"

#: src/sbackup/backupproc.py:535
#, python-format
msgid ""
"Backup is not being started.\n"
"%s"
msgstr ""
"Sicherung wird nicht ausgeführt.\n"
"%s"

#: src/sbackup/pkginfo.py:40
msgid ""
"This is a user friendly backup solution for common desktop needs. If you "
"want to help by submitting bugs, by translating or anything else please "
"visit our website."
msgstr ""
"Eine benutzerfreundliche Backup-Lösung für alltägliche Aufgaben im Desktop-"
"Bereich. Falls Du mithelfen möchtest, z.B. durch das Melden von Fehlern, "
"Übersetzen der Anwendung usw. besuche unsere Webseite."

#: src/sbackup/pkginfo.py:43
msgid "Maintainers:"
msgstr "Betreuer:"

#: src/sbackup/pkginfo.py:47
msgid "Former maintainers:"
msgstr ""

#: src/sbackup/pkginfo.py:51
msgid "Contributors:"
msgstr "Beitragende:"

#: src/sbackup/pkginfo.py:56
msgid "translator-credits"
msgstr ""
"Übersetzer\n"
"\n"
"Launchpad Contributions:\n"
"  Jean-Peer Lorenz https://launchpad.net/~peer.loz"

#: src/sbackup/ar_backend/tar.py:109 src/sbackup/ar_backend/tar.py:167
#: src/sbackup/ar_backend/tar.py:213
msgid "Invalid archive type."
msgstr "Ungültiger Archivtyp."

#: src/sbackup/ar_backend/tar.py:123
#, python-format
msgid "Effective path for `%s` is not available"
msgstr ""

#: src/sbackup/ar_backend/tar.py:294
#, fuzzy, python-format
msgid "Unable to get effective path for `%s`"
msgstr "Auf das Zielverzeichnis konnte nicht zugegriffen werden: %s"

#: src/sbackup/ar_backend/tar.py:338
msgid ""
"For the moment split functionality is not compatible with compress option."
msgstr ""
"Zur Zeit ist das Aufteilen von Sicherungen nicht kompatibel mit "
"komprimierten Sicherungen."

#: src/sbackup/ar_backend/tar.py:347 src/sbackup/ar_backend/tar.py:408
#: src/sbackup/core/profile_handler.py:288
#, python-format
msgid "Unable to change permissions for file '%s'."
msgstr "Zugriffsrechte für Datei '%s' konnten nicht geändert werden."

#: src/sbackup/ar_backend/tar.py:356
#, python-format
msgid "Unable to remove temporary file `%(file)s`: %(error)s"
msgstr "Temporäre Datei `%(file)s` konnte nicht entfernt werden: %(error)s"

#: src/sbackup/ar_backend/tar.py:372
msgid "Launching TAR to make incremental backup."
msgstr "Ausführen von TAR um eine inkrementelle Sicherung zu erstellen."

#: src/sbackup/ar_backend/tar.py:401
msgid "Unable to find the SNAR file to make an incremental backup."
msgstr ""
"Die SNAR-Datei zur Erstellung einer inkrementellen Sicherung wurde nicht "
"gefunden."

#: src/sbackup/ar_backend/tar.py:402
msgid "Falling back to full backup."
msgstr "Es wird auf die Erstellung einer vollen Sicherung zurück gefallen."

#: src/sbackup/ar_backend/tar.py:444
msgid "Launching TAR to make a full backup."
msgstr "Ausführen von TAR um eine vollständige Sicherung zu erstellen."

#: src/sbackup/ar_backend/tar.py:511
msgid "Leading '/' from member names were removed."
msgstr "Führende '/' wurden von den Pfaden im Archiv entfernt."

#: src/sbackup/ar_backend/tar.py:513
msgid "Leading '/' from hard link targets were removed."
msgstr ""
"Führende '/' wurden von den Zielen harter Verknüpfungen im Archiv entfernt."

#: src/sbackup/ar_backend/tar.py:522
#, python-format
msgid "TAR returned a message: %s"
msgstr "TAR gab folgende Mitteilung zurück: %s"

#: src/sbackup/ar_backend/tar.py:524
#, python-format
msgid ""
"TAR returned messages:\n"
"%s"
msgstr ""
"TAR gab folgende Mitteilungen zurück:\n"
"%s"

#: src/sbackup/ar_backend/tar.py:527
msgid "TAR has been finished successfully."
msgstr "TAR wurde erfolgreich beendet."

#: src/sbackup/ar_backend/tar.py:531
#, python-format
msgid "TAR returned a warning: %s"
msgstr "TAR lieferte eine Warnung: %s"

#: src/sbackup/ar_backend/tar.py:533
#, python-format
msgid ""
"TAR returned warnings:\n"
"%s"
msgstr ""
"TAR lieferte Warnungen:\n"
"%s"

#: src/sbackup/ar_backend/tar.py:536
msgid "TAR returned warnings but has been finished successfully."
msgstr "TAR gab Warnungen zurück, konnte aber erfolgreich beendet werden."

#: src/sbackup/ar_backend/tar.py:540
msgid "Unable to finish successfully. TAR terminated with errors."
msgstr ""
"Sicherung war nicht erfolgreich. TAR  wurde auf grund von Fehler vorzeitig "
"beendet."

#: src/sbackup/ar_backend/tar.py:721
#, python-format
msgid "Unable to terminate backend process: %s"
msgstr "Der Backend-Prozess konnte nicht beendet werden: %s"

#: src/sbackup/ar_backend/tar.py:754
msgid "Included"
msgstr "Aufgenommen"

#: src/sbackup/ar_backend/tar.py:754
msgid "Not changed"
msgstr "Nicht geändert"

#: src/sbackup/ar_backend/tar.py:754
msgid "Directory"
msgstr "Verzeichnis"

#: src/sbackup/ar_backend/tar.py:782
msgid "Line must be a string"
msgstr "Zeile muss eine Zeichenkette sein"

#: src/sbackup/ar_backend/tar.py:798
msgid "Dumpdir inconsistancy : 'filename' is empty"
msgstr "Verzeichnis zum Ablegen ist inkonsistent: 'filename' ist leer"

#: src/sbackup/ar_backend/tar.py:811
msgid "Dumpdir inconsistancy : 'control' is empty"
msgstr "Verzeichnis zum Ablegen ist inkonsistent: 'control' ist leer"

#: src/sbackup/ar_backend/tar.py:884
#, python-format
msgid "File '%s' does not exist."
msgstr "Die Datei '%s' existiert nicht."

#: src/sbackup/ar_backend/tar.py:1014 src/sbackup/ar_backend/tar.py:1048
#: src/sbackup/ar_backend/tar.py:1148
msgid "The snarfile header is incomplete."
msgstr "Kopf der SNAR-Datei ist unvollständig."

#: src/sbackup/ar_backend/tar.py:1317
msgid "A SnapshotFile is required"
msgstr "Eine Sicherungsdatei wird benötigt"

#: src/sbackup/ar_backend/tar.py:1417
#, python-format
msgid "Directory does not exist: %s."
msgstr "Das Verzeichnis existiert nicht: %s."

#: src/sbackup/core/ConfigManager.py:330
#, python-format
msgid "Profile settings are being read from file '%s'."
msgstr "Einstellungen des Profils werden gelesen aus Datei '%s'."

#: src/sbackup/core/ConfigManager.py:333
#, python-format
msgid ""
"Profile settings are being set to default values. Configuration file is set "
"to '%s'."
msgstr ""
"Einstellungen des Profils werden auf die Standardwerte gesetzt. "
"Konfigurationsdatei wird auf Datei '%s' festgelegt."

#: src/sbackup/core/ConfigManager.py:479
msgid "Option 'target' is missing, aborting."
msgstr "Angabe der Option 'target' fehlt, Abbruch"

#: src/sbackup/core/ConfigManager.py:581 src/sbackup/core/ConfigManager.py:615
#, python-format
msgid ""
"Unable to evaluate '%(parameter)s' as a dictionary (value got = '%(value)r')."
msgstr ""
"Parameter '%(parameter)s' konnte nicht als assoziatives Feld (Dictionary) "
"ausgewertet werden (übergebener Wert = '%(value)r')."

#: src/sbackup/core/ConfigManager.py:607
#, python-format
msgid ""
"Couldn't evaluate '%(parameter)s' as a dictionary (value got = '%(value)r' )"
msgstr ""
"Parameter '%(parameter)s' konnte nicht als assoziatives Feld (Dictionary) "
"ausgewertet werden (value got = '%(value)r' )"

#: src/sbackup/core/ConfigManager.py:631
msgid "You must provide a dictionary."
msgstr "Ein assoziatives Feld (Dictionary) muss übergeben werden."

#: src/sbackup/core/ConfigManager.py:748
#, python-format
msgid "Log output for [%(profile)s] is directed to file '%(file)s'."
msgstr "Logausgabe für [%(profile)s] wird in Datei '%(file)s' umgeleitet."

#: src/sbackup/core/ConfigManager.py:755
#, python-format
msgid "Log output for [%s] is not directed into a file."
msgstr "Logausgabe für [%s] wird nicht in eine Datei umgeleitet."

#: src/sbackup/core/ConfigManager.py:774
#, python-format
msgid "Unable to rename log file '%(src)s'->'%(dst)s': %(err)s"
msgstr ""
"Protokolldatei konnte nicht umbenannt werden '%(src)s'->'%(dst)s': %(err)s"

#: src/sbackup/core/ConfigManager.py:794
#, python-format
msgid "The config file '%s' couldn't be read!"
msgstr "Konfigurationsdatei '%s' konnte nicht eingelesen werden!"

#: src/sbackup/core/ConfigManager.py:806
#, python-format
msgid "section [%(section)s] in '%(configfile)s' should not exist, aborting"
msgstr ""
"Abschnitt [%(section)s] in '%(configfile)s' sollte nicht existieren, Abbruch"

#: src/sbackup/core/ConfigManager.py:886
msgid "No schedule defined in configuration file. Probing from filesystem."
msgstr ""
"Kein Zeitplan in der Konfiguration definiert. Untersuchung des Dateisystems."

#: src/sbackup/core/ConfigManager.py:979 src/sbackup/core/ConfigManager.py:991
msgid "The config file is not set yet into this ConfigManager"
msgstr ""
"Die Konfigurationsdatei wurde in diesem ConfigManager noch nicht gesetzt"

#: src/sbackup/core/ConfigManager.py:1154
msgid "No receiver set."
msgstr "Kein Empfänger gesetzt."

#: src/sbackup/core/ConfigManager.py:1156
msgid "No SMTP server set."
msgstr "Kein SMTP-Server gesetzt."

#: src/sbackup/core/ConfigManager.py:1159
msgid "Username set but no password specified."
msgstr "Benutzername gesetzt, jedoch kein Passwort festgelegt."

#: src/sbackup/core/ConfigManager.py:1162
msgid "Password set but no username specified."
msgstr "Passwort gesetzt, jedoch kein Benutzername festgelegt."

#: src/sbackup/core/ConfigManager.py:1165
msgid ""
"A certificate and key file is given while SSL option (smtptls=1) is not "
"set.\n"
"Select SSL in order to use Certificate and Key."
msgstr ""
"Es wurde eine Zertifikatsdatei und eine Schlüsseldatei festgelegt, obwohl "
"die Option SSL (smtptls=1) nicht gesetzt ist.\n"
"Aktiviere SSL um ein Zertifikat und einen Schlüssel zu verwenden."

#: src/sbackup/core/ConfigManager.py:1169
msgid ""
"When specifying a SSL certificate or key file, a key file resp. certificate "
"is mandatory."
msgstr ""
"Wenn ein SSL Zertifikat oder eine Schlüsseldatei festgelegt wurde, ist die "
"Angabe einer Schlüsseldatei bzw. eines Zertifikats notwendig."

#: src/sbackup/core/ConfigManager.py:1360
msgid "Default Profile"
msgstr "Standardprofil"

#: src/sbackup/core/ConfigManager.py:1361
msgid "Unknown Profile"
msgstr "unbekanntes Profil"

#: src/sbackup/core/ConfigManager.py:1413
msgid "Unlimited"
msgstr "Unbegrenzt"

#: src/sbackup/core/ConfigManager.py:1414
msgid "100 MiB"
msgstr "100 MiB"

#: src/sbackup/core/ConfigManager.py:1415
msgid "250 MiB"
msgstr "250 MiB"

#: src/sbackup/core/ConfigManager.py:1416
msgid "650 MiB"
msgstr "650 MiB"

#: src/sbackup/core/ConfigManager.py:1417
msgid "2 GiB (FAT16)"
msgstr "2 GiB (FAT16)"

#: src/sbackup/core/ConfigManager.py:1418
msgid "4 GiB (FAT32)"
msgstr "4 GiB (FAT32)"

#: src/sbackup/core/ConfigManager.py:1419 src/sbackup/ui/configgui.py:278
#: src/sbackup/ui/configgui.py:1398
msgid "Custom"
msgstr "Benutzerdefiniert"

#: src/sbackup/core/ConfigManager.py:1421
msgid "MP3 Music"
msgstr "MP3-Musik"

#: src/sbackup/core/ConfigManager.py:1422
msgid "AVI Video"
msgstr "AVI-Videos"

#: src/sbackup/core/ConfigManager.py:1423
#: src/sbackup/core/ConfigManager.py:1424
msgid "MPEG Video"
msgstr "MPEG-Videos"

#: src/sbackup/core/ConfigManager.py:1425
msgid "Matrjoshka Video"
msgstr "Matroska-Videos"

#: src/sbackup/core/ConfigManager.py:1426
msgid "OGG Multimedia container"
msgstr "OGG-Multimedia-Container"

#: src/sbackup/core/ConfigManager.py:1427
msgid "CD Images"
msgstr "CD-Abbilder"

#: src/sbackup/core/RestoreManager.py:80
#, python-format
msgid "File '%s' not found in the backup snapshot files list"
msgstr "Datei '%s' konnte in der Sicherungsdatei nicht gefunden werden."

#: src/sbackup/core/RestoreManager.py:82
#, python-format
msgid ""
"File '%(filename)s' not found in snapshot's [%(snapshotname)s] files list, "
"Skipped."
msgstr ""
"Datei '%(filename)s' wurde nicht in der Dateiliste der Sicherung [%"
"(snapshotname)s] gefunden. Wird übersprungen."

#: src/sbackup/core/SnapshotManager.py:119
#, python-format
msgid "Snapshot '%s' not found "
msgstr "Sicherung '%s' wurde nicht gefunden "

#: src/sbackup/core/SnapshotManager.py:213
#, python-format
msgid "Invalid snapshot `%(name)s` found: Name of snapshot not valid."
msgstr ""
"Ungültige Sicherung `%(name)s` gefunden: Name der Sicherung ist nicht gültig."

#: src/sbackup/core/SnapshotManager.py:216
#, python-format
msgid "Invalid snapshot `%(name)s` found: %(error_cause)s."
msgstr "Ungültige Sicherung `%(name)s` gefunden: %(error_cause)s."

#: src/sbackup/core/SnapshotManager.py:305
#, python-format
msgid "No need to rebase a full snapshot '%s'"
msgstr ""
"Die Datengrundlage einer vollständigen Sicherung '%s' braucht nicht "
"umgestellt werden"

#: src/sbackup/core/SnapshotManager.py:307
#, python-format
msgid ""
"'%(snapshotToRebase)s'  doesn't have 'base' file , it might have been broken "
msgstr ""
"'%(snapshotToRebase)s' enthält keine 'base'-Datei, es ist möglicherweise "
"beschädigt "

#: src/sbackup/core/SnapshotManager.py:312
#, python-format
msgid ""
"Cannot rebase a snapshot on an earlier one : '%(snapshotToRebase)s' <= '%"
"(NewBaseSnapshot)s' "
msgstr ""
"Die Datengrundlage eine Sicherung kann nicht auf eine jüngere Sicherung "
"umgestellt werden: '%(snapshotToRebase)s' <= '%(NewBaseSnapshot)s' "

#: src/sbackup/core/SnapshotManager.py:319
#, python-format
msgid "Rebasing '%(current)s' on '%(base)s'"
msgstr "Ändern der Basis von '%(current)s' auf '%(base)s'"

#: src/sbackup/core/SnapshotManager.py:556
#, python-format
msgid "An error occurred when pulling snapshot '%s'."
msgstr "Während des Lesens der Sicherung '%s' ist ein Fehler aufgetreten."

#: src/sbackup/core/SnapshotManager.py:595
msgid "The snarfile header is incomplete !"
msgstr "Dateikopf der 'snar'-Datei ist unvollständig!"

#: src/sbackup/core/SnapshotManager.py:747
#, python-format
msgid "Base of snapshot '%s' is not set."
msgstr "Ausgangssicherung des Sicherung '%s' ist nicht gesetzt."

#: src/sbackup/core/SnapshotManager.py:784
#, python-format
msgid "Snapshot '%s' is already Full, nothing to do (not changing it to full)."
msgstr ""
"Sicherung '%s' ist bereits eine vollständige Sicherung, es gibt nichts zu "
"tun (wird nicht in eine volle Sicherung umgewandelt)."

#: src/sbackup/core/SnapshotManager.py:824
#, python-format
msgid "Cancelling pull of snapshot '%s'"
msgstr "Zugriff auf Sicherung '%s' abgebrochen"

#: src/sbackup/core/UpgradeManager.py:176
#, python-format
msgid "Unable to remove upgrade stamp: %s"
msgstr "Upgrade-Hinweis konnte nicht entfernt werden: %s"

#: src/sbackup/core/UpgradeManager.py:183
#, python-format
msgid "Upgrading from v1.0 to v1.2: %s"
msgstr "Umstellung von v1.0 auf v1.2: %s"

#: src/sbackup/core/UpgradeManager.py:202
#, python-format
msgid "Upgrading to v1.3: %s"
msgstr "Umstellung auf v1.3: %s"

#: src/sbackup/core/UpgradeManager.py:229
#, python-format
msgid "Upgrading to v1.4: %s"
msgstr "Umstellung auf v1.4: %s"

#: src/sbackup/core/UpgradeManager.py:246
#, python-format
msgid "Upgrading to v1.5: %s"
msgstr "Umstellung auf v1.5: %s"

#: src/sbackup/core/UpgradeManager.py:300
#, python-format
msgid "The SNAR file already exist for snapshot '%s'. It is not overwritten."
msgstr ""
"Die SNAR-Datei existiert bereits für die Sicherung '%s'. Sie wird nicht "
"überschrieben."

#: src/sbackup/core/profile_handler.py:90
msgid "Preparation of backup process"
msgstr "Vorbereitung des Sicherungsvorgangs"

#: src/sbackup/core/profile_handler.py:133
msgid ""
"There are snapshots stored in outdated snapshot formats. Please upgrade them "
"using 'Simple Backup-Restoration' if you want to use them."
msgstr ""
"Es gibt Sicherungen die in einem veralteten Sicherungsformat gespeichert "
"wurden. Bitte diese Sicherungen mit Hilfe der Anwendung 'Simple Backup "
"Wiederherstellung' upgraden um sie zu verwenden."

#: src/sbackup/core/profile_handler.py:142
#, python-format
msgid "Snapshot '%(name)s' is being made."
msgstr "Sicherung '%(name)s' wird erstellt."

#: src/sbackup/core/profile_handler.py:150
#, python-format
msgid "Setting Base to '%(value)s'."
msgstr "Festlegen der Ausgangssicherung auf '%(value)s'."

#: src/sbackup/core/profile_handler.py:159
msgid "Setting packages File."
msgstr "Erstellen der Paket-Liste."

#: src/sbackup/core/profile_handler.py:165
msgid "Problem when setting the packages list: "
msgstr "Problem beim Festlegen der Paketliste: "

#: src/sbackup/core/profile_handler.py:169
msgid "Setting Excludes File."
msgstr "Schreiben der 'Nicht Sichern'-Liste."

#: src/sbackup/core/profile_handler.py:177
#, python-format
msgid "Setting compression format to `%s`"
msgstr "Komprimierung festgelegt auf `%s`"

#: src/sbackup/core/profile_handler.py:184
#, python-format
msgid "Setting size of archive chunks to %s"
msgstr "Größe der Teilarchive festgelegt auf %s"

#: src/sbackup/core/profile_handler.py:191
msgid "Option 'Follow symbolic links' is enabled."
msgstr "Option 'Symbolischen Links folgen' ist aktiviert."

#: src/sbackup/core/profile_handler.py:193
msgid "Option 'Follow symbolic links' is disabled."
msgstr "Option 'Symbolischen Links folgen' ist nicht aktiviert."

#: src/sbackup/core/profile_handler.py:202
msgid "Snapshot is being committed"
msgstr "Sicherung wird geschrieben"

#: src/sbackup/core/profile_handler.py:216
#, python-format
msgid "Error while purging old snapshots: %s"
msgstr "Fehler während des Entfernens alter Sicherungen: %s"

#: src/sbackup/core/profile_handler.py:218
msgid "Backup process finished."
msgstr "Sicherungsvorgang wurde beendet."

#: src/sbackup/core/profile_handler.py:225
msgid "Inspect file system and collect file infos"
msgstr "Untersuchen des Dateisystems und sammeln der Dateiinformationen"

#: src/sbackup/core/profile_handler.py:236
msgid "Summary of backup"
msgstr "Zusammenfassung der Sicherung"

#: src/sbackup/core/profile_handler.py:237
#, python-format
msgid "Number of directories: %s."
msgstr "Anzahl von Verzeichnissen: %s."

#: src/sbackup/core/profile_handler.py:238
#, python-format
msgid "Total number of files: %s."
msgstr "Gesamtanzahl von Dateien: %s."

#: src/sbackup/core/profile_handler.py:239
#, python-format
msgid "Number of symlinks: %s."
msgstr "Anzahl symbolischer Verknüpfungen: %s."

#: src/sbackup/core/profile_handler.py:240
#, python-format
msgid "Number of files included in snapshot: %s."
msgstr "Anzahl der in diesen Schnappschuss eingeschlossenen Dateien: %s."

#: src/sbackup/core/profile_handler.py:241
#, python-format
msgid "Number of new files (also included): %s."
msgstr "Anzahl neuer Dateien (ebenfalls eingeschlossen): %s."

#: src/sbackup/core/profile_handler.py:242
#, python-format
msgid "Number of files skipped in incremental snapshot: %s."
msgstr ""
"Anzahl übersprungener Dateien in diesem inkrementellen Schnappschuss: %s."

#: src/sbackup/core/profile_handler.py:243
#, python-format
msgid "Number of items forced to be excluded: %s."
msgstr "Anzahl von Elementen, die ausgeschlossen werden mussten: %s."

#: src/sbackup/core/profile_handler.py:244
#, python-format
msgid "Number of items to be excluded by config: %s."
msgstr ""
"Anzahl von Elementen, die auf Grund von Benutzereinstellungen ausgeschlossen "
"wurden: %s."

#: src/sbackup/core/profile_handler.py:245
#, python-format
msgid "Maximum free size required is '%s'."
msgstr "Maximal benötigter freier Speicherplatz ist '%s'."

#: src/sbackup/core/profile_handler.py:251
#, python-format
msgid "Available disk size is '%s'."
msgstr "Verfügbarer Speicherplatz ist '%s'."

#: src/sbackup/core/profile_handler.py:253
#, python-format
msgid ""
"Not enough free space in the target directory for the planned backup (free: %"
"(freespace)s, required: %(neededspace)s)."
msgstr ""
"Für die geplante Sicherung ist nicht genügend freier Speicher im "
"Zielverzeichnis verfügbar (frei: %(freespace)s, benötigt: %(neededspace)s)."

#: src/sbackup/core/profile_handler.py:273
msgid "Unable to copy log. File access is not initialized."
msgstr ""
"Protokolldatei konnte nicht kopiert werden. Dateizugriff wurde nicht "
"initialisiert."

#: src/sbackup/core/profile_handler.py:291
#, python-format
msgid "Unable to copy log file: %s"
msgstr "Protokolldatei konnte nicht kopiert werden: %s"

#: src/sbackup/core/profile_handler.py:293
msgid "Unable to find logfile to copy into snapshot."
msgstr "Logdatei wurde nicht gefunden."

#: src/sbackup/core/profile_handler.py:295
msgid "No snapshot to copy logfile."
msgstr "Keine Sicherung vorhanden um das Log zu kopieren."

#: src/sbackup/core/profile_handler.py:302
msgid "Processing of profile was canceled on user request\n"
msgstr ""
"Die Bearbeitung des Sicherungsprofils wurde auf Wunsch des Benutzers "
"abgebrochen\n"

#: src/sbackup/core/profile_handler.py:319
msgid "Processing of profile successfully finished (no errors)\n"
msgstr ""
"Die Bearbeitung des Sicherungsprofils wurde erfolgreich beendet (keine "
"Fehler)\n"

#: src/sbackup/core/profile_handler.py:325
#, python-format
msgid "Processing of profile failed with error: %s\n"
msgstr "Die Bearbeitung des Sicherungsprofils ist fehlgeschlagen. Fehler: %s\n"

#: src/sbackup/core/profile_handler.py:336
#, python-format
msgid "Backup destination: %s"
msgstr "Sicherungsziel: %s"

#: src/sbackup/core/profile_handler.py:340
msgid "Unable to find destination directory."
msgstr "Zielverzeichnis konnte nicht gefunden werden."

#: src/sbackup/core/profile_handler.py:359
msgid "Unable to find destination directory even after retry."
msgstr ""
"Zielverzeichnis konnte auch nach einem weiteren Versuch nicht gefunden "
"werden."

#: src/sbackup/core/profile_handler.py:360
#: src/sbackup/core/profile_handler.py:367
#, python-format
msgid "Target directory '%(target)s' does not exist."
msgstr "Zielverzeichnis '%(target)s' existiert nicht."

#: src/sbackup/core/profile_handler.py:373
#, python-format
msgid "Unable to access destination: %s"
msgstr "Auf das Zielverzeichnis konnte nicht zugegriffen werden: %s"

#: src/sbackup/core/snapshot.py:132
msgid "Snapshot is inconsistent: __name is not set "
msgstr "Schnappschuss ist inkonsistent: __name ist nicht festgelegt "

#: src/sbackup/core/snapshot.py:143
#, python-format
msgid "Name of snapshot '%s' doesn't match requirement"
msgstr "Name der Sicherung '%s' erfüllt Anforderung nicht"

#: src/sbackup/core/snapshot.py:237
msgid "Snapshot is inconsistent: __snapshotpath is not set "
msgstr "Schnappschuss ist inkonsistent: __snapshotpath ist nicht festgelegt "

#: src/sbackup/core/snapshot.py:324
#, python-format
msgid ""
"The snapshot compression format is supposed to be '%s' but the corresponding "
"well named file wasn't found"
msgstr ""
"Das Format der Sicherung sollte '%s' sein, jedoch wurde ein entsprechend "
"benannte Datei nicht gefunden"

#: src/sbackup/core/snapshot.py:348
#, python-format
msgid ""
"%(file)s doesn't contain valid value. Ignoring incomplete or non - backup "
"directory. "
msgstr ""

#: src/sbackup/core/snapshot.py:484
#, python-format
msgid "Name of Snapshot not valid : %s"
msgstr "Name der Sicherung ist ungültig: %s"

#: src/sbackup/core/snapshot.py:502
#, python-format
msgid "Name of base not valid : %s"
msgstr "Name der Ausgangssicherung ist ungültig: %s"

#: src/sbackup/core/snapshot.py:548
#, python-format
msgid "Invalid name of snapshot: %s"
msgstr "Ungültiger Name der Sicherung: %s"

#: src/sbackup/core/snapshot.py:552
msgid "Unable to read mandatory file `ver`"
msgstr "Die notwendige Datei `ver` konnte nicht gelesen werden"

#: src/sbackup/core/snapshot.py:597
msgid "Base name must be set for incremental backup."
msgstr ""
"Für eine inkrementelle Sicherung muss der Name einer Ausgangssicherung "
"festgelegt sein."

#: src/sbackup/fs_backend/_fuse_fam.py:106
msgid "Initializing FUSE File Access Manager."
msgstr "Initialisierung des FUSE File Access Manager."

#: src/sbackup/fs_backend/_fuse_fam.py:129
msgid "Terminating FUSE File Access Manager."
msgstr "Beendigung des FUSE File Access Manager."

#: src/sbackup/fs_backend/_fuse_utils.py:44
#: src/sbackup/fs_backend/_gio_utils.py:107
msgid "SFTP"
msgstr "SFTP"

#: src/sbackup/fs_backend/_fuse_utils.py:45
#: src/sbackup/fs_backend/_gio_utils.py:108
msgid "SSH"
msgstr "SSH"

#: src/sbackup/fs_backend/_fuse_utils.py:46
#: src/sbackup/fs_backend/_gio_utils.py:109
msgid "FTP"
msgstr "FTP"

#: src/sbackup/fs_backend/_gio_fam.py:100
msgid "Initializing GIO File Access Manager."
msgstr "Initialisierung des GIO Dateizugriffs."

#: src/sbackup/fs_backend/_gio_fam.py:114
msgid "Terminating GIO File Access Manager."
msgstr "Beendigung des GIO Dateizugriffs."

#: src/sbackup/fs_backend/_gio_utils.py:110
msgid "NFS"
msgstr "NFS"

#: src/sbackup/fs_backend/_gio_utils.py:111
msgid "SMB"
msgstr "SMB"

#: src/sbackup/fs_backend/_gio_utils.py:191
msgid "Unable to get mount: path is not mounted"
msgstr "Mountpunkt konnte nicht ermittelt werden: Pfad ist nicht eingebunden"

#: src/sbackup/fs_backend/_gio_utils.py:193
msgid "Unable to get mount: path not found when mounting (is probably local)"
msgstr ""
"Mountpunkt konnte nicht ermittelt werden: Einzubindener Pfad wurde nicht "
"gefunden (ist wahrscheinlich lokal)"

#: src/sbackup/fs_backend/_gio_utils.py:195
#: src/sbackup/fs_backend/_gio_utils.py:198
#, python-format
msgid "Unable to get mount: %s"
msgstr "Mountpunkt konnte nicht ermittelt werden: %s"

#: src/sbackup/fs_backend/_gio_utils.py:245
msgid "Unable to un-set mount flag: path is not stored"
msgstr ""
"Mountflag konnte nicht zurück gesetzt werden: Pfad wurde nicht gesichert"

#: src/sbackup/fs_backend/_gio_utils.py:347
msgid "Path is already mounted."
msgstr "Pfad ist bereits eingebunden."

#: src/sbackup/fs_backend/_gio_utils.py:370
msgid "Path is not mounted."
msgstr "Pfad ist nicht eingebunden."

#: src/sbackup/fs_backend/_gio_utils.py:837
msgid "test specified path for existence using GIO"
msgstr "Testen des festgelegten Pfades auf Existenz mittels GIO"

#: src/sbackup/fs_backend/_gio_utils.py:850
msgid "Create testdir"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:855
msgid "Test testfile for existence"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:862
msgid "Create file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:869
msgid "Re-read test file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:881
msgid "Remove file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:883
msgid "Remove dir"
msgstr ""

#: src/sbackup/fuse_plugins/__init__.py:160
#, python-format
msgid "Unable to import plugin `%(plugin_name)s`: %(error_cause)s "
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:117
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of ftp "
"sites: %s"
msgstr ""
"Konnte externe Anwendung 'curlftpfs' nicht finden. Diese wird zur Handhabung "
"von FTP Sites benötigt: %s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:127
#: src/sbackup/fuse_plugins/sftpFuseFAM.py:124
#, python-format
msgid "Couldn't mount '%(server)s' into '%(mountpoint)s' : %(error)s"
msgstr ""
"'%(server)s' konnte nicht in '%(mountpoint)s' eingebunden werden: %(error)s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:141
msgid "FTP schema is: ftp://user:pass@server/anything"
msgstr ""
"Schema für FTP ist: ftp://benutzername:passwort@servername.com/"
"ein_beliebiger_pfad"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:188
#, python-format
msgid ""
"Error matching the schema 'ftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Beim Auflösen des FTP-Adressformates 'ftp://user:pass@server/anything' ist "
"ein Fehler aufgetreten: '%s' (auf den Servernamen muss zwingend '/' folgen)"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:114
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of sftp "
"sites: %s"
msgstr ""
"Für die Verwendung von SFTP-Zielen wird das externe Programm 'curlftpfs' "
"benötigt. Das Programm 'curlftpfs' wurde nicht gefunden: %s"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:138
msgid "SFTP schema is like : sftp://user:pass@server/anything"
msgstr "Das Format für SFTP-Adressen lautet: sftp://user:pass@server/anything"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:185
#, python-format
msgid ""
"Error matching the schema 'sftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Beim Auflösen des SFTP-Adressformates 'sftp://user:pass@server/anything' ist "
"ein Fehler aufgetreten: '%s' (auf den Servernamen muss zwingend '/' folgen)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:81
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com/home/' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Beim Auflösen des SSH-Adressformates 'ssh://user:pass@example.com/home/' ist "
"ein Fehler aufgetreten: '%s' (auf den Servernamen muss zwingend '/' folgen)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:137
#, python-format
msgid ""
"The sshfs command '%(command)s' didn't perform normally. Output => %"
"(erroroutput)s "
msgstr ""
"Das sshfs-Kommando '%(command)s' wurde nicht korrekt ausgeführt. Ausgabe => %"
"(erroroutput)s "

#: src/sbackup/fuse_plugins/sshFuseFAM.py:142
msgid "SSH schema is like : ssh://user:pass@example.com:33/home/user/backup/"
msgstr ""
"Schema für SSH ist: ssh://benutzername:passwort@servername.com:portnummer/"
"ein_beliebiger_pfad"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:158
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com:21/home/' with '%"
"s' (The '/' after server is mandatory)"
msgstr ""
"Fehler beim Prüfen des Schema 'ssh://benutzername:passwort@servername.com:"
"portnummer/ein_beliebiger_pfad' mit '%s' (Der Schrägstrich '/' nach der "
"Serverangabe ist notwendig)."

#: src/sbackup/ui/GladeWindow.py:209
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target."
msgstr ""
"Bitte stellen sie sicher, dass das fehlende Verzeichnis existiert (z.B. "
"durch Einhängen einer externen Festplatte) oder ändern sie den Wert für das "
"Zielverzeichnis entsprechend."

#: src/sbackup/ui/GladeWindow.py:210
#, python-format
msgid ""
"While accessing the backup destination following error occurred:\n"
"\n"
"%s."
msgstr ""

#: src/sbackup/ui/GladeWindow.py:211 src/sbackup/ui/restoregui.py:816
#: src/sbackup/ui/restoregui.py:870
msgid "Simple Backup error"
msgstr "Simple Backup Fehler"

#: src/sbackup/ui/GladeWindow.py:212
msgid "Unable to access backup destination"
msgstr "Auf das Sicherungsziel konnte nicht zugegriffen werden"

#: src/sbackup/ui/configgui.py:124
msgid "Scheduled backups are available for Administrator users only."
msgstr "Sicherungen nach Zeitplan sind nur für Administratoren möglichen."

#: src/sbackup/ui/configgui.py:134 src/sbackup/ui/configgui.py:143
msgid "Name"
msgstr "Name"

#: src/sbackup/ui/configgui.py:151
msgid "File Type"
msgstr "Dateityp"

#: src/sbackup/ui/configgui.py:186
msgid "Enable"
msgstr "Profil verwenden"

#: src/sbackup/ui/configgui.py:187
msgid "Profile Name"
msgstr "Name des Profils"

#: src/sbackup/ui/configgui.py:284
#, python-format
msgid ""
"Invalid or empty regular expression ('%s') found in configuration. Removed."
msgstr ""
"Ungültigen oder leeren Regulären Ausdruck ('%s') in der Konfiguration "
"gefunden. Wurde entfernt."

#: src/sbackup/ui/configgui.py:291
#, python-format
msgid ""
"Invalid or empty regular expressions found\n"
"in configuration file:\n"
"'%s'\n"
"\n"
"These expressions are not used and were\n"
"removed from the configuration."
msgstr ""
"Ungültige oder leere Reguläre Ausdrücke in\n"
"der Konfiguration gefunden:\n"
"'%s'\n"
"\n"
"Diese Ausdrücke werden nicht benutzt und wurden\n"
"aus der Konfiguration entfernt."

#: src/sbackup/ui/configgui.py:547
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target to an existing one."
msgstr ""
"Bitte stellen Sie sicher, dass das fehlende Verzeichnis existiert (z. B. "
"indem sie eine externe Festplatte einbinden) oder ändern sie das festgelegte "
"Zielverzeichnis auf ein existierendes Verzeichnis."

#: src/sbackup/ui/configgui.py:548
#, python-format
msgid ""
"Backup target `%s` does not exist.\n"
"\n"
"Attention: The target will be set to the default value. Check this on the "
"destination settings page before saving the configuration."
msgstr ""
"Ziel der Sicherung `%s` existiert nicht.\n"
"\n"
"Achtung: Das Sicherungsziel wird auf den Standardwert zurück gesetzt. Prüfen "
"sie dies bitte im Reiter 'Zielverzeichnis' bevor sie die Änderungen "
"speichern."

#: src/sbackup/ui/configgui.py:550
msgid "Unable to open backup target"
msgstr "Zielverzeichnis der Sicherung konnte nicht geöffnet werden"

#: src/sbackup/ui/configgui.py:621
#, python-format
msgid "Current profile: %s"
msgstr "Aktuelles Profil: %s"

#: src/sbackup/ui/configgui.py:712
msgid "Already included item !"
msgstr "Eintrag ist bereits enthalten!"

#: src/sbackup/ui/configgui.py:727
msgid "Already excluded item !"
msgstr "Eintrag ist bereits ausgeschlossen!"

#: src/sbackup/ui/configgui.py:739 src/sbackup/ui/configgui.py:1438
msgid "Empty expression. Please enter a valid regular expression."
msgstr ""
"Leeres Suchmuster. Bitte geben Sie einen gültigen regulären Ausdruck als "
"Suchmuster an."

#: src/sbackup/ui/configgui.py:754 src/sbackup/ui/configgui.py:1452
msgid "Provided regular expression is not valid."
msgstr "Der angegebene reguläre Ausdruck ist nicht gültig."

#: src/sbackup/ui/configgui.py:762
msgid "Empty filename or path. Please enter a valid filename or path."
msgstr ""
"Leerer Dateiname oder Pfad. Bitte geben Sie einen gültigen Dateinamen oder "
"Pfad an."

#: src/sbackup/ui/configgui.py:771
msgid ""
"It seems the path you entered does not exists. Do you want to add this "
"incorrect path?"
msgstr ""
"Der angegebene Pfad scheint nicht zu existieren. Soll dieser inkorrekte Pfad "
"hinzugefügt werden?"

#: src/sbackup/ui/configgui.py:800
msgid "Save configuration as..."
msgstr "Konfiguration speichern unter..."

#: src/sbackup/ui/configgui.py:867
msgid ""
"A backup process is now executed in the background.\n"
"\n"
"You can monitor the progress of the backup by means of the status indicator "
"displayed in the notification area."
msgstr ""
"Ein Sicherungsvorgang wird im Hintergrund ausgeführt.\n"
"\n"
"Sie können den Fortschritt der Sicherung mit Hilfe des Statusindikators im "
"Benachrichtigungsbereich überwachen."

#: src/sbackup/ui/configgui.py:868
msgid "Backup process started"
msgstr "Sicherungsvorgang gestartet"

#: src/sbackup/ui/configgui.py:873 src/sbackup/ui/configgui.py:1641
#: src/sbackup/ui/configgui.py:2011
#, python-format
msgid "Unexpected dialog response: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:947
msgid "Include file..."
msgstr "Datei hinzufügen..."

#: src/sbackup/ui/configgui.py:954 src/sbackup/ui/configgui.py:1009
msgid "All files"
msgstr "Alle Dateien"

#: src/sbackup/ui/configgui.py:974
msgid "Include folder..."
msgstr "Ordner hinzufügen..."

#: src/sbackup/ui/configgui.py:1002
msgid "Exclude file..."
msgstr "Datei ausschliessen..."

#: src/sbackup/ui/configgui.py:1025
msgid "Exclude folder..."
msgstr "Ordner ausschliessen..."

#: src/sbackup/ui/configgui.py:1290
msgid "Configuration has changed"
msgstr "Konfiguration wurde geändert"

#: src/sbackup/ui/configgui.py:1291
msgid ""
"There are unsaved modifications. Please save the configuration or revert "
"these changes before testing the mail settings."
msgstr ""
"Es gibt ungesicherte Änderungen. Bitte sichere die Konfiguration oder mache "
"die Änderungen rückgängig um die E-Mail-Einstellungen zu testen."

#: src/sbackup/ui/configgui.py:1297 src/sbackup/ui/configgui.py:1303
msgid "Test mail settings"
msgstr "Testen der Einstellungen für E-Mail"

#: src/sbackup/ui/configgui.py:1298
msgid "The test failed with following output:"
msgstr "Der Test schlug mit folgender Ausgabe fehl:"

#: src/sbackup/ui/configgui.py:1304
msgid "The test was successful."
msgstr "Der Test war erfolgreich."

#: src/sbackup/ui/configgui.py:1387 src/sbackup/ui/configgui.py:1430
#, python-format
msgid ""
"The given expression contains unsupported characters ('%s'). Currently it is "
"not possible to use these characters in exclude expressions."
msgstr ""
"Der übergebene Ausdruck enthält nicht unterstützte Zeichen ('%s'). Es ist "
"zur Zeit nicht möglich diese Zeichen in 'Nicht Sichern'-Ausdrücken zu "
"verwenden."

#: src/sbackup/ui/configgui.py:1390 src/sbackup/ui/configgui.py:1433
msgid "Unsupported character"
msgstr "Nicht unterstütztes Zeichen"

#: src/sbackup/ui/configgui.py:1601
msgid "Connect to remote destination"
msgstr ""

#: src/sbackup/ui/configgui.py:1649
msgid "Mount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1660
msgid "Unable to mount host"
msgstr ""

#: src/sbackup/ui/configgui.py:1669
msgid "Umount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1679
msgid "Unable to unmount host"
msgstr ""

#: src/sbackup/ui/configgui.py:1709
msgid "Perfom tests on remote host"
msgstr ""

#: src/sbackup/ui/configgui.py:1722
msgid "Unable to access remote destination"
msgstr ""

#: src/sbackup/ui/configgui.py:1726
msgid "All tests passed"
msgstr ""

#: src/sbackup/ui/configgui.py:1883
msgid ""
"The given name of the new profile is empty. Please enter a valid profile "
"name."
msgstr ""
"Der übergebene Profilname ist leer. Bitte einen gültigen Profilnamen "
"eingeben."

#: src/sbackup/ui/configgui.py:1884 src/sbackup/ui/configgui.py:1891
msgid "Profile name not valid"
msgstr "Profilname nicht gültig"

#: src/sbackup/ui/configgui.py:1890
msgid ""
"The given name of the new profile already exists. Please enter another name."
msgstr ""
"Der übergebene Profilname wird bereits verwendet. Bitte einen anderen "
"Profilnamen eingeben."

#: src/sbackup/ui/configgui.py:1892
msgid "Renaming of profiles is not supported."
msgstr "Das Umbenennen von Profilen wird nicht unterstützt."

#: src/sbackup/ui/configgui.py:1922
#, python-format
msgid ""
"<b>Delete configuration profile?</b>\n"
"\n"
"You are trying to remove a configuration profile. You will not be able to "
"restore it. If you are not sure, use the 'enable|disable' functionality "
"instead.\n"
"\n"
"Do you really want to delete the profile '%(name)s'?"
msgstr ""

#: src/sbackup/ui/configgui.py:1995
#, python-format
msgid ""
"<big><b>Set default values for current profile?</b></big>\n"
"This will restore the default values for the profile currently edited: '%"
"s'.\n"
"\n"
"These predefined settings are recommended for most users. Check whether they "
"are appropriate for your use before saving the changed configuration."
msgstr ""
"<big><b>Standardwerte für das aktuelle Profil setzen?</b></big>\n"
"Es werden die Standardwerte für das derzeitig bearbeitete Profil '%s' wieder "
"hergestellt.\n"
"\n"
"Diese vordefinierten Einstellungen sind für die meisten Anwender empfohlen. "
"Vor dem Speichern der geänderten Konfiguration sollte überprüft werden, ob "
"diese Einstellungen passend für den konkreten Anwendungsfall sind."

#: src/sbackup/ui/configgui.py:2005
msgid "Default settings are being applied."
msgstr "Standardwerte werden gesetzt."

#: src/sbackup/ui/configgui.py:2030
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be removed. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2044
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be disabled. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2058
msgid ""
"<b>No backup profile found.</b>\n"
"\n"
"No default profile was found. You are probably running Simple Backup for the "
"first time. A backup profile using default values was created.\n"
"\n"
"Please modify the settings according to your needs and save the "
"configuration in order to use it."
msgstr ""

#: src/sbackup/ui/configgui.py:2071
msgid "Please select a profile."
msgstr ""

#: src/sbackup/ui/indicator_core.py:116
msgid "there was a problem initializing the pynotify module"
msgstr "Es gab ein Problem beim Laden des pynotify Modul."

#: src/sbackup/ui/indicator_core.py:145 src/sbackup/ui/indicator_core.py:192
#: src/sbackup/ui/indicator_core.py:220 src/sbackup/ui/indicator_core.py:242
#, python-format
msgid "Connection to notification-daemon failed: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:305
msgid "Simple Backup"
msgstr "Simple Backup"

#: src/sbackup/ui/indicator_core.py:311
msgid "Show message windows"
msgstr "Dialogfenster anzeigen"

#: src/sbackup/ui/indicator_core.py:353 src/sbackup/ui/indicator_core.py:813
msgid "Cancel Backup"
msgstr "Sicherung abbrechen"

#: src/sbackup/ui/indicator_core.py:415
msgid "Connection to D-Bus service lost."
msgstr ""

#: src/sbackup/ui/indicator_core.py:440
msgid "re-connection was successful"
msgstr ""

#: src/sbackup/ui/indicator_core.py:479
msgid ""
"No backup in progress and connection to D-Bus service lost. Simple Backup "
"Indicator is being terminated."
msgstr ""

#: src/sbackup/ui/indicator_core.py:508
msgid "Starting backup session"
msgstr "Sicherungsvorgang wird gestartet"

#: src/sbackup/ui/indicator_core.py:522
msgid "Ending Backup Session"
msgstr "Sicherungsvorgang wird beendet"

#: src/sbackup/ui/indicator_core.py:535
msgid ""
"There are snapshots with old snapshot format. Please upgrade these using the "
"Restoration tool if you want to use them."
msgstr ""

#: src/sbackup/ui/indicator_core.py:537
#, python-format
msgid "Unknown D-Bus event `%s` received."
msgstr ""

#: src/sbackup/ui/indicator_core.py:578
msgid "Try again"
msgstr "Erneut versuchen"

#: src/sbackup/ui/indicator_core.py:648
msgid ""
"Attempt of starting another instance of Simple Backup while this one is "
"already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:658
#, python-format
msgid "Unable to present window: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:674
#, python-format
msgid "Do you really want to cancel the backup process of profile '%s'?"
msgstr ""
"Soll der Sicherungsvorgang des Profils '%s' wirklich abgebrochen werden?"

#: src/sbackup/ui/indicator_core.py:678
msgid "Cancel running backup process?"
msgstr "Laufenden Sicherungsvorgang abbrechen?"

#: src/sbackup/ui/indicator_core.py:695
msgid "Unable to cancel backup process."
msgstr ""

#: src/sbackup/ui/indicator_core.py:757
#, python-format
msgid "Profile: %s"
msgstr "Profil: %s"

#: src/sbackup/ui/indicator_core.py:758
#, python-format
msgid "Size of backup: %s"
msgstr "Größe der Sicherung: %s"

#: src/sbackup/ui/indicator_core.py:759
#, python-format
msgid "%.1f%% processed"
msgstr "%.1f%% erledigt"

#: src/sbackup/ui/indicator_core.py:760
#, python-format
msgid "Remaining time: %s"
msgstr "Verbleibende Zeit: %s"

#: src/sbackup/ui/indicator_core.py:806
#, python-format
msgid "PID of backup process being canceled: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:811
#, python-format
msgid "Unable to send signal to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:812
#, python-format
msgid "Sending signal using gksu to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:816
#, python-format
msgid "Sending signal using gksu to process %s failed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:851
#, python-format
msgid ""
"<b>Unable to find specified target directory</b>\n"
"\n"
"The specified target directory '%s' was not found.\n"
"\n"
msgstr ""

#: src/sbackup/ui/indicator_core.py:852
#, python-format
msgid ""
"You can try to use the specified target again or cancel the profile "
"execution. The specified destination is automatically used in %s seconds."
msgstr ""
"Sie können erneut versuchen das festgelegt Sicherungsziel zu verwenden oder "
"den Sicherungsvorgang abbrechen. Die festgelegte Sicherung wird in %s "
"Sekunden automatisch erneut verwendet."

#: src/sbackup/ui/indicator_core.py:879 src/sbackup/ui/indicator_core.py:921
#: src/sbackup/ui/indicator_core.py:939 src/sbackup/ui/indicator_core.py:947
#: src/sbackup/ui/indicator_core.py:964
msgid "unknown"
msgstr "unbekannt"

#: src/sbackup/ui/indicator_core.py:900
msgid "less than 1 minute"
msgstr "weniger als 1 Minute"

#: src/sbackup/ui/indicator_core.py:902
msgid "about 2 minutes"
msgstr "ungefähr 2 Minuten"

#: src/sbackup/ui/indicator_core.py:904
msgid "about 3 minutes"
msgstr "ungefähr 3 Minuten"

#: src/sbackup/ui/indicator_core.py:906
msgid "about 4 minutes"
msgstr "ungefähr 4 Minuten"

#: src/sbackup/ui/indicator_core.py:908
msgid "about 5 minutes"
msgstr "ungefähr 5 Minuten"

#: src/sbackup/ui/indicator_core.py:910
#, python-format
msgid "about %.0f minutes"
msgstr "ungefähr %.0f Minuten"

#: src/sbackup/ui/indicator_core.py:920
msgid "In progress"
msgstr "Vorgang läuft"

#: src/sbackup/ui/indicator_core.py:930 src/sbackup/ui/indicator_core.py:956
msgid "finished"
msgstr "Fertiggestellt"

#: src/sbackup/ui/indicator_core.py:938
msgid "Progress: unknown"
msgstr "Fortschritt: unbekannt"

#: src/sbackup/ui/indicator_core.py:945
msgid "Backup is being prepared"
msgstr "Sicherung wird vorbereitet"

#: src/sbackup/ui/indicator_core.py:946
msgid "Collecting file info"
msgstr "Dateiinformationen sammeln"

#: src/sbackup/ui/indicator_core.py:955
msgid "Progress: canceled"
msgstr "Fortschritt: abgebrochen"

#: src/sbackup/ui/indicator_core.py:977
msgid "An error occurred"
msgstr "Ein Fehler trat auf"

#: src/sbackup/ui/indicator_core.py:1010
msgid "Another `Simple Backup Indicator` is already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:1034
#, python-format
msgid "Initialization of GTK+ failed: %s"
msgstr ""

#: src/sbackup/ui/misc.py:293
#, python-format
msgid "%s   (Administrator mode)"
msgstr "%s (Administrator-Modus)"

#: src/sbackup/ui/misc.py:323
#, python-format
msgid ""
"An uncaught error occurred. Close this message window and restart the "
"application.\n"
"\n"
"Please report this error on https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"
msgstr ""

#: src/sbackup/ui/misc.py:324
msgid "Sorry, this should not have happened"
msgstr ""

#: src/sbackup/ui/restoregui.py:77
msgid "Restore..."
msgstr "Wiederherstellen..."

#: src/sbackup/ui/restoregui.py:78 data/ui/sbackup-restore.glade:845
msgid "Restore as..."
msgstr "Wiederherstellen unter..."

#: src/sbackup/ui/restoregui.py:79
msgid "Revert..."
msgstr "Rückgängig machen..."

#: src/sbackup/ui/restoregui.py:80 data/ui/sbackup-restore.glade:877
msgid "Revert as..."
msgstr "Rückgängig machen unter..."

#: src/sbackup/ui/restoregui.py:127 data/ui/sbackup-restore.glade:111
#: data/ui/sbackup-restore.glade:591
msgid "n.a."
msgstr "k.A."

#: src/sbackup/ui/restoregui.py:145 src/sbackup/ui/restoregui.py:162
msgid "Snapshots"
msgstr "Sicherungen"

#: src/sbackup/ui/restoregui.py:146
msgid "Version"
msgstr "Version"

#: src/sbackup/ui/restoregui.py:153 data/ui/sbackup-config.glade:4063
msgid "Path"
msgstr "Pfad"

#: src/sbackup/ui/restoregui.py:158
msgid "State"
msgstr "Status"

#: src/sbackup/ui/restoregui.py:186
msgid "Connect to destination..."
msgstr ""

#: src/sbackup/ui/restoregui.py:210
msgid "Specified path does not exist"
msgstr ""

#: src/sbackup/ui/restoregui.py:256
msgid "Ready"
msgstr "Fertig"

#: src/sbackup/ui/restoregui.py:370
msgid "No destination specified"
msgstr ""

#: src/sbackup/ui/restoregui.py:387
msgid "Choose a source folder"
msgstr "Wählen Sie einen Quellordner"

#: src/sbackup/ui/restoregui.py:404
msgid "Reading snapshots..."
msgstr "Einlesen der Sicherungen..."

#: src/sbackup/ui/restoregui.py:427
msgid "Reading directory content..."
msgstr "Einlesen des Verzeichnisinhalts..."

#: src/sbackup/ui/restoregui.py:457 src/sbackup/ui/restoregui.py:495
msgid "Loading ..."
msgstr "Lade ..."

#: src/sbackup/ui/restoregui.py:538
msgid "Reading files from snapshot..."
msgstr "Einlesen der Dateien der Sicherung..."

#: src/sbackup/ui/restoregui.py:551
msgid "Simple Backup restore error"
msgstr ""

#: src/sbackup/ui/restoregui.py:552
msgid "An error occurred while reading snapshot"
msgstr "Während des Einlesen der Sicherung ist ein Fehler aufgetreten"

#: src/sbackup/ui/restoregui.py:556
msgid "This snapshot seems empty."
msgstr "Diese Sicherung scheint leer zu sein."

#: src/sbackup/ui/restoregui.py:575
msgid "Reading backup snapshot..."
msgstr "Lese Sicherungsschnappschuss"

#: src/sbackup/ui/restoregui.py:598
msgid "No backups found for this day."
msgstr "Keine Sicherungen für dieses Datum gefunden."

#: src/sbackup/ui/restoregui.py:613
#, python-format
msgid ""
"Snapshot version is not supported (Only %(supportedversion)s is supported). "
"Version '%(currentversion)s' found. You should upgrade it. "
msgstr ""

#: src/sbackup/ui/restoregui.py:642
msgid "Select restore location"
msgstr "Ort für die Wiederherstellung auswählen"

#: src/sbackup/ui/restoregui.py:661
#, python-format
msgid "Do you really want to revert '%s'?"
msgstr "Sollen die Änderungen an '%s' wirklich rückgängig gemacht werden?"

#: src/sbackup/ui/restoregui.py:675
msgid "Select revert location"
msgstr "Auswahl eines Verzeichnisses um Änderungen rückgängig zu machen"

#: src/sbackup/ui/restoregui.py:684
#, python-format
msgid "Do you really want to revert '%(source)s' to '%(dir)s'?"
msgstr ""
"\"Sollen die Änderungen an '%(source)s' wirklich nach '%(dir)s' rückgängig "
"gemacht werden?\""

#: src/sbackup/ui/restoregui.py:762
msgid "Reading snapshot..."
msgstr "Einlesen der Sicherung..."

#: src/sbackup/ui/restoregui.py:781
msgid "The version of the snapshot is greater than the supported one!"
msgstr "Die Version der Sicherung ist neuer als die unterstützten Versionen!"

#: src/sbackup/ui/restoregui.py:791
#, python-format
msgid ""
"Upgrading the selected snapshot `%s` might corrupt your backup for this "
"particular point in time. You cannot undo this operation. It is recommended "
"to create a copy of the snapshot directory before upgrading it. The upgrade "
"process takes a while - be patient.\n"
"\n"
"Are you sure that you want to upgrade the snapshot now?"
msgstr ""

#: src/sbackup/ui/restoregui.py:795
msgid "Do you really want to upgrade snapshot?"
msgstr ""

#: src/sbackup/ui/restoregui.py:799
#, python-format
msgid "Upgrade snapshot '%s'"
msgstr ""

#: src/sbackup/ui/restoregui.py:808
msgid "Snapshot was sucessfully upgraded."
msgstr ""

#: src/sbackup/ui/restoregui.py:809
msgid "Simple Backup upgrade"
msgstr ""

#: src/sbackup/ui/restoregui.py:810
msgid "Upgrade finished"
msgstr ""

#: src/sbackup/ui/restoregui.py:813
#, python-format
msgid ""
"While attempting to upgrade snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:816
msgid "Unable to upgrade snapshot"
msgstr "Upgrade der Sicherung konnte nicht durchgeführt werden"

#: src/sbackup/ui/restoregui.py:837
#, python-format
msgid "Do you really want to rebase '%(current)s' on '%(base)s' ?"
msgstr ""
"Möchten Sie wirklich die Basis von '%(current)s' auf '%(base)s' ändern?"

#: src/sbackup/ui/restoregui.py:855
#, python-format
msgid ""
"Deleting the selected snapshot `%s` will remove your backup for this "
"particular point in time. You cannot undo this operation.\n"
"\n"
"Are you sure that you want to remove the snapshot permanently?"
msgstr ""

#: src/sbackup/ui/restoregui.py:859
msgid "Do you really want to remove snapshot?"
msgstr ""

#: src/sbackup/ui/restoregui.py:868
#, python-format
msgid ""
"While attempting to delete snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:871
msgid "Unable to delete snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:877
#, python-format
msgid ""
"The selected snapshot '%s' is referenced by more recent snapshots as base "
"snapshot.\n"
"\n"
"Remove all of these child snapshots in order to delete this snapshot."
msgstr ""

#: src/sbackup/ui/restoregui.py:879
msgid "Unable to remove snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:887
msgid "Termination in progress"
msgstr ""

#: src/sbackup/ui/restoregui.py:903 src/sbackup/ui/restoregui.py:909
#: src/sbackup/ui/restoregui.py:915 src/sbackup/ui/restoregui.py:921
#: data/ui/sbackup-restore.glade:1039
msgid "Simple Backup restoration"
msgstr ""

#: src/sbackup/ui/restoregui.py:904 src/sbackup/ui/restoregui.py:910
msgid "<b>Restoring of selected files</b>"
msgstr "<b>Wiederherstellung der ausgewählten Dateien</b>"

#: src/sbackup/ui/restoregui.py:905
#, python-format
msgid "Restoring of <tt>'%s'</tt> is in progress."
msgstr "Die Wiederherstellung von <tt>'%s'</tt> wird durchgeführt."

#: src/sbackup/ui/restoregui.py:906
#, python-format
msgid "Restoring of <tt>'%s'</tt> was successful."
msgstr "Wiederherstellung von <tt>'%s'</tt> war erfolgreich."

#: src/sbackup/ui/restoregui.py:907
#, python-format
msgid ""
"Restoring of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:911
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Wiederherstellung von <tt>'%(source)s'</tt>\n"
"nach <tt>'%(dirname)s'</tt> wird ausgeführt."

#: src/sbackup/ui/restoregui.py:912
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""
"Wiederherstellung von <tt>'%(source)s'</tt>\n"
"nach <tt>'%(dirname)s'</tt> war erfolgreich."

#: src/sbackup/ui/restoregui.py:913
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:916 src/sbackup/ui/restoregui.py:922
msgid "<b>Reverting selected files</b>"
msgstr "<b>Änderungen an ausgewählten Dateien werden rückgängig gemacht</b>"

#: src/sbackup/ui/restoregui.py:917
#, python-format
msgid "Reverting of <tt>'%s'</tt> is in progress.\n"
msgstr "Rückgängig machen von <tt>'%s'</tt> wird durchgeführt.\n"

#: src/sbackup/ui/restoregui.py:918
#, python-format
msgid "Reverting of <tt>'%s'</tt> was successful."
msgstr "Rückgängig machen von <tt>'%s'</tt> war erfolgreich."

#: src/sbackup/ui/restoregui.py:919
#, python-format
msgid ""
"Reverting of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:923
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Rückgängig machen von <tt>'%(source)s'</tt>\n"
"nach <tt>'%(dirname)s'</tt> wird durchgeführt."

#: src/sbackup/ui/restoregui.py:924
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""
"Rückgängig machen von <tt>'%(source)s'</tt>\n"
"nach <tt>'%(dirname)s'</tt> war erfolgreich."

#: src/sbackup/ui/restoregui.py:925
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/util/__init__.py:439
#, python-format
msgid "%(mb)d MiB %(kb)d KiB"
msgstr ""

#: src/sbackup/util/__init__.py:442
#, python-format
msgid "%(mb)d MiB %(kb)d KiB %(bytes)d"
msgstr "%(mb)d MiB %(kb)d KiB %(bytes)d"

#: src/sbackup/util/__init__.py:447
#, python-format
msgid "%(mb)d MB %(kb)d kB"
msgstr ""

#: src/sbackup/util/__init__.py:450
#, python-format
msgid "%(mb)d MB %(kb)d kB %(bytes)d"
msgstr "%(mb)d MB %(kb)d kB %(bytes)d"

#: src/sbackup/util/filecollect.py:335
#, python-format
msgid "File '%(file)s' does not exist."
msgstr "Datei '%(file)s' existiert nicht."

#: src/sbackup/util/filecollect.py:348
#, python-format
msgid "File '%(file)s' is not accessable with error '%(error)s'."
msgstr ""
"Auf Datei '%(file)s' kann nicht zugegriffen werden. Fehler '%(error)s'."

#: src/sbackup/util/filecollect.py:364
#, python-format
msgid "File '%(file)s' cannot be opened for read access. Operation timed out."
msgstr ""
"Datei '%(file)s' kann nicht zum Lesen geöffnet werden. Zeitüberschreitung "
"beim Öffnen."

#: src/sbackup/util/filecollect.py:368
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access with error '%(error)s'."
msgstr ""
"Datei '%(file)s' kann nicht zum Lesen geöffnet werden. Fehler '%(error)s'."

#: src/sbackup/util/filecollect.py:385
#, python-format
msgid "File '%(file)s' is backup's target directory."
msgstr "Pfad '%(file)s' ist das Zielverzeichnis der Sicherung."

#: src/sbackup/util/filecollect.py:394
#, python-format
msgid "Symbolic link '%(path)s' -> '%(ln_target)s' is a circular symlink."
msgstr ""
"Symbolische Verknüpfung '%(path)s' -> '%(ln_target)s' erzeugt einen "
"Kreisbezug."

#: src/sbackup/util/filecollect.py:414
#, python-format
msgid ""
"File '%(file)s' exceeds maximum file size ( %(filesize)s > %(maxsize)s)."
msgstr ""
"Datei '%(file)s' überschreitet die maximale Dateigröße  ( %(filesize)s > %"
"(maxsize)s)."

#: src/sbackup/util/filecollect.py:424
#, python-format
msgid "File '%(file)s' matches regular expression '%(regex)s'."
msgstr "Pfad '%(file)s' erfüllt den Regulären Ausdruck '%(regex)s'."

#: src/sbackup/util/filecollect.py:430
#, python-format
msgid "Path '%(file)s' defined in excludes list."
msgstr "Pfad '%(file)s' ist in der 'Nicht Sichern'-Liste vorhanden."

#: src/sbackup/util/filecollect.py:501
#, python-format
msgid "Error while checking directory '%(dir)s': %(error)s."
msgstr "Fehler beim Testen des Verzeichnisses '%(dir)s': %(error)s."

#: src/sbackup/util/filecollect.py:552
msgid "Empty regular expression found. Skipped."
msgstr "Leerer Regulärer Ausdruck gefunden. Wird ausgelassen."

#: src/sbackup/util/filecollect.py:558
#, python-format
msgid "Invalid regular expression ('%s') found. Skipped."
msgstr "Ungültiger Regulärer Ausdruck ('%s') gefunden. Wird ausgelassen."

#: src/sbackup/util/filecollect.py:577
msgid "No directories to backup defined."
msgstr "Keine Verzeichnisse zum Sichern festgelegt."

#: src/sbackup/util/lock.py:76
msgid "Another application instance is already running."
msgstr ""

#: src/sbackup/util/lock.py:78
msgid "Invalid lock file found. Is being removed."
msgstr "Ungültige Lock-Datei gefunden. Wird entfernt."

#: src/sbackup/util/lock.py:128 src/sbackup/util/lock.py:142
#, python-format
msgid "Unable to remove lock file: %s"
msgstr "Lock-Datei konnte nicht entfernt werden: %s"

#: src/sbackup/util/lock.py:132 src/sbackup/util/lock.py:144
msgid "Unable to remove lock file: File not found."
msgstr ""

#: data/ui/sbackup-config.glade:11
msgid "Exclude file type"
msgstr "Dateityp ausschließen"

#: data/ui/sbackup-config.glade:26
msgid "Standard file types:"
msgstr "Standard Dateitypen:"

#: data/ui/sbackup-config.glade:63
msgid "Custom defined file type with "
msgstr "Selbstdefinierter Dateityp mit "

#: data/ui/sbackup-config.glade:93
msgid "file extension"
msgstr "Dateierweiterung"

#: data/ui/sbackup-config.glade:163
msgid "Exclude by regex"
msgstr "Ausschließen mittels Suchmuster (regulärer Ausdruck)"

#: data/ui/sbackup-config.glade:180
msgid ""
"Enter a regular expression.\n"
"Files and directories matching this expression will be excluded from backup."
msgstr ""
"Geben Sie einen regulären Ausdruck an.\n"
"Dateien und Ordner, die diesem Ausdruck entsprechen, werden von der "
"Sicherung ausgeschlossen."

#: data/ui/sbackup-config.glade:257
msgid "Enter the remote target to include"
msgstr "Geben Sie einen entfernten Ordner/Datei an, der gesichert werden soll"

#: data/ui/sbackup-config.glade:278
msgid "Choose a plugin to get the format"
msgstr "Wählen Sie ein Plugin um das Format zu bestimmen"

#: data/ui/sbackup-config.glade:295
msgid "Choose your plugin :"
msgstr "Wählen Sie Ihr Plugin:"

#: data/ui/sbackup-config.glade:367
msgid "Check"
msgstr "Prüfen"

#: data/ui/sbackup-config.glade:479
msgid ""
"<b>Save changed configuration?</b>\n"
"\n"
"The configuration has changed. If you want to use the modified "
"configuration, you have to save it.\n"
"\n"
"Do you want to save the configuration now?"
msgstr ""
"<b>Geänderte Konfiguration speichern?</b>\n"
"\n"
"Die Konfiguration wurde geändert. Soll die geänderte Konfiguration verwendet "
"werden, so muss sie gespeichert werden.\n"
"\n"
"Soll die Konfiguration jetzt gespeichert werden?"

#: data/ui/sbackup-config.glade:572
msgid "Configuration Profile Manager"
msgstr "Profilverwaltung"

#: data/ui/sbackup-config.glade:594
msgid ""
"You can set up multiple configuration profiles which are processed when "
"performing a backup. Each profile has its own set of settings including "
"backup destination, backup format etc."
msgstr ""

#: data/ui/sbackup-config.glade:613
msgid "<b>Configuration profiles</b>"
msgstr "Einstellungsprofile"

#: data/ui/sbackup-config.glade:785
msgid ""
"<b>Add new profile</b>\n"
"\n"
"Please enter the new profile name and set the according checkbox in order to "
"enable resp. disable it."
msgstr ""
"<b>Hinzufügen eines Profils</b>\n"
"\n"
"Bitte den Namen des neuen Profils angeben und festlegen ob das Profil aktiv "
"bzw. inaktiv sein soll."

#: data/ui/sbackup-config.glade:810
msgid "Profile name"
msgstr "Profilname"

#: data/ui/sbackup-config.glade:836
msgid "Enabled"
msgstr "Aktiviert"

#: data/ui/sbackup-config.glade:915 data/ui/sbackup-restore.glade:7
msgid "Simple Backup Suite"
msgstr "Simple Backup Suite"

#: data/ui/sbackup-config.glade:932
msgid "_File"
msgstr "_Datei"

#: data/ui/sbackup-config.glade:971
msgid "Save as..."
msgstr "Speichern unter..."

#: data/ui/sbackup-config.glade:1008
msgid "_Tools"
msgstr "_Werkzeuge"

#: data/ui/sbackup-config.glade:1016 data/ui/sbackup-config.glade:1140
msgid "Set default settings..."
msgstr "Standartwerte setzen..."

#: data/ui/sbackup-config.glade:1037
msgid "Profile Manager..."
msgstr "Profilverwaltung..."

#: data/ui/sbackup-config.glade:1058
msgid "Make backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1079
msgid "_Help"
msgstr "_Hilfe"

#: data/ui/sbackup-config.glade:1087
msgid "_View all help topics"
msgstr "_Hilfethemen"

#: data/ui/sbackup-config.glade:1103
msgid "_About this program"
msgstr "Info"

#: data/ui/sbackup-config.glade:1154
msgid "Reload configuration"
msgstr "Einstellungen neu laden"

#: data/ui/sbackup-config.glade:1168
msgid "Save configuration"
msgstr "Konfiguration speichern"

#: data/ui/sbackup-config.glade:1191
msgid "Make a backup now"
msgstr "Sicherung jetzt erstellen"

#: data/ui/sbackup-config.glade:1214
msgid "Open the profile manager"
msgstr "Profilverwaltung öffnen"

#: data/ui/sbackup-config.glade:1215
msgid "Profiles"
msgstr "Profile"

#: data/ui/sbackup-config.glade:1253
msgid ""
"none\n"
"gzip\n"
"bzip2"
msgstr ""
"none\n"
"gzip\n"
"bzip2"

#: data/ui/sbackup-config.glade:1271
msgid ""
"Compression format\n"
"Default is set to uncompressed backups."
msgstr ""

#: data/ui/sbackup-config.glade:1290 data/ui/sbackup-config.glade:1403
msgid ""
"You can split the snapshot to make backup on some filesystem that doesn't "
"support large files. This functionality can not be used if the backup is "
"compressed."
msgstr ""
"Die Sicherung kann in kleinere Dateien aufgeteilt werden, um Sicherung auf "
"Dateisystemen abzulegen, die keine beliebig großen Dateien unterstützen. "
"Diese Funktion kann nicht in Verbindung mit komprimierten Sicherungen "
"verwendet werden."

#: data/ui/sbackup-config.glade:1330
msgid "Do a full backup at least once every"
msgstr "Vollständige Sicherung mindestens alle"

#: data/ui/sbackup-config.glade:1339
msgid ""
"Unless a set number of days has passed since the last full backup, "
"incremental backups will be done."
msgstr ""
"Bis zur festgelegten Anzahl an Tagen seit der letzten vollständigen "
"Sicherung werden inkrementelle Sicherungen erstellt."

#: data/ui/sbackup-config.glade:1356 data/ui/sbackup-config.glade:3015
msgid "days"
msgstr "Tage"

#: data/ui/sbackup-config.glade:1370
msgid " "
msgstr " "

#: data/ui/sbackup-config.glade:1380 data/ui/sbackup-config.glade:2323
msgid "MiB"
msgstr "MiB"

#: data/ui/sbackup-config.glade:1405
msgid "Split backup archives into several chunks"
msgstr ""

#: data/ui/sbackup-config.glade:1438 data/ui/sbackup-config.glade:2627
#: data/ui/sbackup-config.glade:2863 data/ui/sbackup-config.glade:3111
#: data/ui/sbackup-config.glade:3856
msgid "Help"
msgstr "Hilfe"

#: data/ui/sbackup-config.glade:1442 data/ui/sbackup-config.glade:2632
msgid "Show help topic about configuration GUI."
msgstr "Hilfe zu den Einstellungen mittels grafischer Oberfläche zeigen."

#: data/ui/sbackup-config.glade:1475
msgid "General"
msgstr "Allgemein"

#: data/ui/sbackup-config.glade:1495
msgid "<b>Included files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1570 data/ui/sbackup-config.glade:1818
#: data/ui/sbackup-config.glade:2023 data/ui/sbackup-config.glade:2181
msgid "Remove"
msgstr "Entfernen"

#: data/ui/sbackup-config.glade:1620 data/ui/sbackup-config.glade:1868
msgid "Add File"
msgstr "Datei hinzufügen"

#: data/ui/sbackup-config.glade:1671 data/ui/sbackup-config.glade:1919
msgid "Add Directory"
msgstr "Ordner hinzufügen"

#: data/ui/sbackup-config.glade:1718
msgid "Include"
msgstr "Zu sichernde Daten"

#: data/ui/sbackup-config.glade:1739
msgid "<b>Excluded files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1952
msgid "Paths"
msgstr "Pfade"

#: data/ui/sbackup-config.glade:2074 data/ui/sbackup-config.glade:2232
msgid "Add"
msgstr "Hinzufügen"

#: data/ui/sbackup-config.glade:2109
msgid "File types"
msgstr "Dateitypen"

#: data/ui/sbackup-config.glade:2267
msgid ""
"Regular\n"
"Expressions"
msgstr ""
"Reguläre\n"
"Ausdrücke"

#: data/ui/sbackup-config.glade:2288
msgid "Do not backup files bigger than"
msgstr "Keine Dateien sichern, die größer sind als"

#: data/ui/sbackup-config.glade:2344
msgid "Follow symbolic links"
msgstr "Symbolischen Verknüpfungen folgen"

#: data/ui/sbackup-config.glade:2373
msgid "Others"
msgstr "Sonstige"

#: data/ui/sbackup-config.glade:2400
msgid "Exclude"
msgstr "Nicht Sichern"

#: data/ui/sbackup-config.glade:2421
msgid "Use default backup directory"
msgstr "Standardverzeichnis für Sicherungen verwenden"

#: data/ui/sbackup-config.glade:2446
msgid "/var/backup"
msgstr "/var/backup"

#: data/ui/sbackup-config.glade:2472
msgid "Use custom local backup directory"
msgstr "Benutzerdefiniertes lokales Sicherungs-Verzeichnis verwenden"

#: data/ui/sbackup-config.glade:2500
msgid "Select the destination folder"
msgstr "Zielverzeichnis auswählen"

#: data/ui/sbackup-config.glade:2526
msgid "Use a remote site"
msgstr ""

#: data/ui/sbackup-config.glade:2568
msgid "ssh://username:password@example.com/remote/dir/"
msgstr "ssh://benutzername:kennwort@beispiel.com/entfernt/ordner/"

#: data/ui/sbackup-config.glade:2580
msgid "Connect..."
msgstr ""

#: data/ui/sbackup-config.glade:2672
msgid ""
"<i>The selected destination is not writeable with current permissions. "
"Backups cannot be stored there.</i>"
msgstr ""
"<i>Das ausgewählte Zielverzeichnis ist mit den derzeitigen Zugriffsrechten "
"nicht beschreibbar. Sicherungen können dort nicht abgelegt werden.</i>"

#: data/ui/sbackup-config.glade:2704
msgid "Destination"
msgstr "Zielverzeichnis"

#: data/ui/sbackup-config.glade:2725
msgid "_No scheduled backups"
msgstr "_Keine geplanten/automatischen Sicherungen"

#: data/ui/sbackup-config.glade:2742
msgid "_Simple"
msgstr "_Einfach"

#: data/ui/sbackup-config.glade:2747
msgid ""
"This option will make a backup not at a precise time but when the computer "
"is up. "
msgstr ""

#: data/ui/sbackup-config.glade:2761
msgid "_Custom"
msgstr "_Benutzerdefiniert"

#: data/ui/sbackup-config.glade:2781
msgid "Do backups"
msgstr "Sicherungen erstellen"

#: data/ui/sbackup-config.glade:2798
msgid "Cron time definition"
msgstr "Cron-Ausdruck"

#: data/ui/sbackup-config.glade:2834
msgid ""
"hourly\n"
"daily\n"
"weekly\n"
"monthly"
msgstr ""
"Stündlich\n"
"Täglich\n"
"Wöchentlich\n"
"Monatlich"

#: data/ui/sbackup-config.glade:2867
msgid "Show help topic about scheduled backups."
msgstr "Hilfe zu geplanten Sicherungen anzeigen."

#: data/ui/sbackup-config.glade:2905
msgid ""
"<small>You can configure scheduled (i.e. automated) backups either simply by "
"choosing an interval or precisely by a custom cron definition. See \"man 5 "
"crontab\" for cron time definition format.</small>"
msgstr ""
"<small>Geplante/Zeitgesteuerte (d.h. automatische) Sicherungen können durch "
"das einfache Auswählen des gewünschten Intervalles oder durch die Festlegung "
"eines benutzerdefinierten Cron-Ausdruckes konfiguriert werden. Siehe \"man 5 "
"crontab\" für das Format der Cron-Ausdrücke.</small>"

#: data/ui/sbackup-config.glade:2937
msgid "Schedule"
msgstr "Zeitplan"

#: data/ui/sbackup-config.glade:2955
msgid "Enable purging of old backups"
msgstr ""

#: data/ui/sbackup-config.glade:2983
msgid "Simple cut-off: Erase all backups older than"
msgstr "Einfaches Entfernen: Löschen aller Sicherungen älter als"

#: data/ui/sbackup-config.glade:3039
msgid "Logarithmic"
msgstr "Logarithmisches Entfernen"

#: data/ui/sbackup-config.glade:3065
msgid ""
"<small>Keep progressively less backups into the past:\n"
"Keep all backups from yesterday\n"
"Keep one backup per day from last week\n"
"Keep one backup per week from last month\n"
"Keep one backup per month from last year\n"
"Keep one backup per quarter from 2nd last year\n"
"Keep one backup per year further into past</small>"
msgstr ""

#: data/ui/sbackup-config.glade:3116
msgid "Show help topic about purging of backups."
msgstr "Hilfe zum Aufräumen von Sicherungen anzeigen."

#: data/ui/sbackup-config.glade:3152
msgid ""
"<small>Note: Additionally all incomplete and irrecoverable backups will be "
"erased as long as there is a complete backup snapshot made later on.</small>"
msgstr ""
"<small>Hinweis: Darüber hinaus werden alle unvollständigen und nicht "
"wiederherstellbaren Sicherungen gelöscht, sofern eine vollständige Sicherung "
"eines späteren Zeitpunktes verfügbar ist.</small>"

#: data/ui/sbackup-config.glade:3184
msgid "Purging"
msgstr "Aufräumen"

#: data/ui/sbackup-config.glade:3186
msgid "Cleaning old backups"
msgstr "Alte Sicherungen säubern"

#: data/ui/sbackup-config.glade:3209
msgid "<b>Logging</b>"
msgstr "<b>Protokollieren</b>"

#: data/ui/sbackup-config.glade:3234
msgid "Debug is just for debbuging purpose.Info is the recommended setting"
msgstr ""
"Die empfohlene Einstellung ist 'Info'. Die Einstellung 'Debug' ist für "
"Entwicklerzwecke vorgesehen."

#: data/ui/sbackup-config.glade:3236
msgid "Log level"
msgstr "Protokollierungsstufe"

#: data/ui/sbackup-config.glade:3248
msgid "Debug is just for debbuging purpose. Info is the recommended setting."
msgstr ""
"'Debug' dient der Fehlerbehebung. Die empfohlene Einstellung is 'Info'."

#: data/ui/sbackup-config.glade:3274
msgid "Log file directory"
msgstr "Verzeichnis der Protokolldatei"

#: data/ui/sbackup-config.glade:3320
msgid "<b>Report by mail</b>"
msgstr "<b>Bericht per E-Mail</b>"

#: data/ui/sbackup-config.glade:3350
msgid "From"
msgstr "Absender"

#: data/ui/sbackup-config.glade:3361
msgid "SMTP server"
msgstr "SMTP-Server"

#: data/ui/sbackup-config.glade:3375 data/ui/sbackup-config.glade:4053
msgid "Port"
msgstr "Portnummer"

#: data/ui/sbackup-config.glade:3391
msgid "To"
msgstr "An"

#: data/ui/sbackup-config.glade:3478
msgid "Use a username and password"
msgstr "Benutzername und Kennwort verwenden"

#: data/ui/sbackup-config.glade:3508 data/ui/sbackup-config.glade:4075
msgid "Username"
msgstr "Benutzername"

#: data/ui/sbackup-config.glade:3540 data/ui/sbackup-config.glade:4124
msgid "Password"
msgstr "Kennwort"

#: data/ui/sbackup-config.glade:3584
msgid "Use secure connection"
msgstr "Sichere Verbindung benutzen"

#: data/ui/sbackup-config.glade:3615
msgid "Choose a certificate file"
msgstr "Zertifikatsdatei auswählen"

#: data/ui/sbackup-config.glade:3629
msgid "Key"
msgstr "Schlüssel"

#: data/ui/sbackup-config.glade:3645
msgid "Certificate"
msgstr "Zertifikat"

#: data/ui/sbackup-config.glade:3659
msgid "Choose a key file"
msgstr "Schlüsseldatei auswählen"

#: data/ui/sbackup-config.glade:3731
msgid "Test _mail settings"
msgstr "Testen der _Mail Einstellungen"

#: data/ui/sbackup-config.glade:3772
msgid "Report"
msgstr "Berichte"

#: data/ui/sbackup-config.glade:3811
msgid "Warning"
msgstr "Achtung"

#: data/ui/sbackup-config.glade:3860
msgid "Show help topic about default values."
msgstr "Zeige das Hilfethema zu Standardwerten."

#: data/ui/sbackup-config.glade:3893
msgid "_Set"
msgstr "_Setzen"

#: data/ui/sbackup-config.glade:3935
msgid "Set remote target"
msgstr ""

#: data/ui/sbackup-config.glade:3957
msgid "Type of service"
msgstr ""

#: data/ui/sbackup-config.glade:3997
msgid "Hostname"
msgstr ""

#: data/ui/sbackup-config.glade:4027
msgid "<b>Details</b>"
msgstr ""

#: data/ui/sbackup-config.glade:4157
msgid "Show password"
msgstr ""

#: data/ui/sbackup-config.glade:4217
msgid "_Connect"
msgstr ""

#: data/ui/sbackup-config.glade:4293
msgid "Make a full backup"
msgstr ""

#: data/ui/sbackup-restore.glade:38
msgid "<b>Backup destination</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:52
msgid "Displays the manual in Help browser."
msgstr "Zeigt das Handbuch im Hilfebrowser."

#: data/ui/sbackup-restore.glade:85
msgid "Default profile's destination"
msgstr ""

#: data/ui/sbackup-restore.glade:90
msgid ""
"Set backup destination to the place specified in default configuration "
"profile"
msgstr ""

#: data/ui/sbackup-restore.glade:137
msgid "Custom destination"
msgstr ""

#: data/ui/sbackup-restore.glade:143
msgid ""
"Select a backup destination on your harddisk or at a remote site. Press "
"Apply when ready."
msgstr ""

#: data/ui/sbackup-restore.glade:184
msgid "Choose a destination folder. For remote sites fill URI in text field"
msgstr ""

#: data/ui/sbackup-restore.glade:201
msgid "Change destination to path specified in left text field"
msgstr ""

#: data/ui/sbackup-restore.glade:266
msgid "Select a bold printed date to get the list of backups for that date."
msgstr ""

#: data/ui/sbackup-restore.glade:310
msgid "Available backups"
msgstr ""

#: data/ui/sbackup-restore.glade:398
msgid "Delete snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:436
msgid "Delete"
msgstr "Entfernen"

#: data/ui/sbackup-restore.glade:475
msgid "Upgrade snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:515
msgid "Upgrade"
msgstr "Version umstellen"

#: data/ui/sbackup-restore.glade:553
msgid "<b>Snapshot History</b>"
msgstr "<b>Verlauf der Sicherungen</b>"

#: data/ui/sbackup-restore.glade:579
msgid "Current base:"
msgstr ""

#: data/ui/sbackup-restore.glade:620
msgid "Select a snapshot on this list to rebase on it."
msgstr ""
"Wählen Sie aus dieser Liste eine Sicherung aus um diese als "
"Ausgangssicherung zu verwenden."

#: data/ui/sbackup-restore.glade:695
msgid "Rebase"
msgstr "Ausgangssicherung umstellen"

#: data/ui/sbackup-restore.glade:746
msgid "<b>Snapshot Management</b>"
msgstr "<b>Verwaltung von Sicherungen</b>"

#: data/ui/sbackup-restore.glade:785
msgid "Snapshot details"
msgstr ""

#: data/ui/sbackup-restore.glade:850 data/ui/sbackup-restore.glade:867
msgid "Restores files/directories from the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:862
msgid "Restore"
msgstr "Wiederherstellen"

#: data/ui/sbackup-restore.glade:882 data/ui/sbackup-restore.glade:901
msgid ""
"Restores files/directories recursively starting from the base snapshot to "
"the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:896
msgid "Revert"
msgstr "Änderungen rückgängig machen"

#: data/ui/sbackup-restore.glade:939
msgid "<b>Restoration Management</b>"
msgstr "<b>Wiederherstellung</b>"

#: data/ui/sbackup-restore.glade:964
msgid "<b>Export Management</b>"
msgstr "<b>Export-Verwaltung</b>"

#: data/ui/sbackup-restore.glade:1056
msgid "Headline"
msgstr "Überschrift"

#: data/ui/sbackup-restore.glade:1069
msgid "Content"
msgstr "Inhalt"

#~ msgid "(Not So) Simple Backup error"
#~ msgstr "(Not So) Simple Backup Fehler"

#~ msgid "sbackup restoration"
#~ msgstr "sbackup-Wiederherstellung"

#~ msgid "Import of module 'pynotify' failed with error: %s."
#~ msgstr "Import des Moduls 'pynotify' fehlgeschlagen. Fehler: %s."

#~ msgid "Failed to set the gid to 'admin' one :"
#~ msgstr "Fehler beim Setzen der GID zur Gruppe 'admin':"

#~ msgid "Setting split size."
#~ msgstr "Setzen der Größe geteilter Sicherungen."

#~ msgid "Preparation of backup is done. Archive is being created."
#~ msgstr "Vorbereitung der Sicherung abgeschlossen. Archiv wird erstellt."

#~ msgid "Target not writeable: "
#~ msgstr "Ziel ist nicht beschreibbar: "

#~ msgid "Invalid value of the target directory : "
#~ msgstr "Ungültiger Wert für das Zielverzeichnis: "

#~ msgid ""
#~ "Got a non valid snapshot '%(name)s' due to name convention : %"
#~ "(error_cause)s "
#~ msgstr ""
#~ "Ungültige Sicherung '%(name)s' gefunden, fehlerhaftes Namensformat: %"
#~ "(error_cause)s "

#~ msgid "Deleting snapshot: '%(snp)s'."
#~ msgstr "Löschen der Sicherung: '%(snp)s'."

#~ msgid "Cannot downgrade other format than 'gzip' to 1.4"
#~ msgstr ""
#~ "Nur Sicherungen im Format 'gzip' können auf die ältere Version 1.4 "
#~ "umgestellt werden"

#~ msgid ""
#~ "A backup run is initiated in the background.\n"
#~ "The process id is: %s."
#~ msgstr ""
#~ "Ein Sicherungsvorgang wurde als Hintergrundprozess gestartet.\n"
#~ "Die Prozess-Id lautet: %s."

#~ msgid "Test on '%s' didn't succeed!"
#~ msgstr "Der Test mit '%s' schlug fehl!"

#~ msgid "'%d' test(s) didn't succeed !"
#~ msgstr "'%d' Test(s) sind fehlgeschlagen!"

#~ msgid "Test Succeeded !"
#~ msgstr "Test war erfolgreich!"

#~ msgid "Target directory is writable."
#~ msgstr "Zielverzeichnis ist nicht schreibbar."

#~ msgid ""
#~ "Please change target directory and test writability of the target "
#~ "directory by pressing \"Test\" button on the right."
#~ msgstr ""
#~ "Bitte ändern Sie das Zielverzeichnis und testen Sie, ob Schreiben möglich "
#~ "ist durch Anklicken des 'Test'-Buttons."

#~ msgid "remove"
#~ msgstr "Entfernen"

#~ msgid "disable"
#~ msgstr "Deaktivieren"

#~ msgid ""
#~ "You can't %s the Default Profile. Please use it if you need only one "
#~ "profile."
#~ msgstr ""
#~ "Das Standardprofil kann nicht %s werden. Bitte nutze es, falls du nur ein "
#~ "Profil benötigst."

#~ msgid ""
#~ "The program is going to be terminated. Please make sure the missing "
#~ "directory exists (e.g. by mounting an external disk) or change the "
#~ "specified target in '(Not So) Simple Backup-Configuration' tool and "
#~ "restart this application."
#~ msgstr ""
#~ "Die Anwendung wird beendet. Bitte sicherstellen, dass das fehlende "
#~ "Verzeichnis existiert (z.B. durch das Anschließen einer externen "
#~ "Festplatte) oder das festgelegte Zielverzeichnis in '(Not So) Simple "
#~ "Backup-Einstellungen' ändern. Anschließend diese Anwendung erneut starten."

#~ msgid "An error occurred during initialization:"
#~ msgstr "Während der Initialisierung ist ein Fehler aufgetreten:"

#~ msgid "Actual base : <b>%s</b>"
#~ msgstr "Derzeitige Ausgangssicherung: <b>%s</b>"

#~ msgid "(Not So) Simple Backup restoration error"
#~ msgstr "(Not So) Simple Backup Wiederherstellung Fehler"

#~ msgid ""
#~ "Are you sure that you want to remove snapshot '%s'?\n"
#~ "\n"
#~ "This may take a while - be patient."
#~ msgstr ""
#~ "Soll die Sicherung '%s' wirklich entfernt werden?\n"
#~ "\n"
#~ "Dieser Vorgang kann einige Zeit in Anspruch nehmen - bitte geduldig sein."

#~ msgid "The mandatory 'ver' file doesn't exist in [%s]."
#~ msgstr "Die notwendige Datei 'ver' existiert nicht in [%s]."

#~ msgid "Command 'env' failed."
#~ msgstr "Befehl 'env' ist fehlgeschlagen."

#~ msgid "Unable to make a proper backup. TAR was terminated."
#~ msgstr ""
#~ "Es konnte keine fehlerfreie Sicherung erstellt werden. TAR wurde "
#~ "vorzeitig abgebrochen."

#~ msgid ""
#~ "This dialogue helps you to manage your configuration profiles. You can "
#~ "set up a number of configuration profiles that will be run by sbackup "
#~ "when it starts the backup. Thanks to this you can configure backups of "
#~ "different kinds to different places."
#~ msgstr ""
#~ "Dieser Dialog hilft bei der Verwaltung von Profilen. Es können eine "
#~ "Vielzahl von Profilen mit jeweils eigenen und unabhängigen "
#~ "Konfigurationen erstellt werden, die während des Sicherungsvorgangs "
#~ "abgearbeitet werden. Mit Hilfe von Profilen können unterschiedliche Arten "
#~ "von Sicherungen in unterschiedliche Zielverzeichnisse erstellt werden."

#~ msgid ""
#~ "Choose the compression format that you want to apply.\n"
#~ "Default is <b>gzip</b>."
#~ msgstr ""
#~ "Wählen Sie das gewünschte Kompressionsverfahren.\n"
#~ "Standardmäßig wird mit <b>gzip</b> komprimiert."

#~ msgid ""
#~ "Split the backup archive into small pieces (value in MiB).\n"
#~ "<b>Important note:</b> This functionality is not compatible with "
#~ "compressed backups!"
#~ msgstr ""
#~ "Das Sicherungsarchiv in Teilstücke aufteilen (Wert in MiB).\n"
#~ "<b>Wichtiger Hinweis:</b> Diese Funktion kann nicht in Verbindung mit "
#~ "komprimierten Sicherungen verwendet werden!"

#~ msgid "<b>Remote paths</b>"
#~ msgstr "<b>Entfernte Pfade</b>"

#~ msgid ""
#~ "<b>Using this window you can manage your backups, restore them, revert an "
#~ "existing directory to a previous state...</b>"
#~ msgstr ""
#~ "<b>Mit diesem Dialog können Sicherungen verwaltet werden, Dateien wieder "
#~ "hergestellt, vorherige Zustände von vorhandenen Verzeichnissen "
#~ "restauriert werden...</b>"

#~ msgid ""
#~ "Select the directory in which you have your backups.\n"
#~ "You can use a <b>local</b> or <b>distant</b> directory depending on the "
#~ "plugins you have installed."
#~ msgstr ""
#~ "Wählen Sie ein Verzeichnis mit ihren Sicherungen.\n"
#~ "Abhängig von den installierten Plugins können Sie <b>lokale</b> oder "
#~ "<b>entfernte</b> Verzeichnisse auswählen."

#~ msgid "Use the default directory : "
#~ msgstr "Standard-Verzeichnis benutzen: "

#~ msgid "Use a custom directory : "
#~ msgstr "Benutzerdefiniertes Verzeichnis benutzen: "

#~ msgid "<b>Snapshots directory</b>"
#~ msgstr "Verzeichnis mit Sicherungen"

#~ msgid "Restore As"
#~ msgstr "Wiederherstellen unter"

#~ msgid "Revert As"
#~ msgstr "Änderungen rückgängig machen unter"

#~ msgid "Choose to upgrade your snapshot if needed."
#~ msgstr "Wählen um die Version Ihrer Sicherung umzustellen, falls notwendig."

#~ msgid "Actual base :"
#~ msgstr "Derzeitige Ausgangssicherung:"

#~ msgid "Select a snapshot and change its base to another snapshot."
#~ msgstr ""
#~ "Ausgangssicherung einer inkrementellen Sicherung auf eine andere "
#~ "Sicherung umstellen."

#~ msgid "BackupManager created "
#~ msgstr "BackupManager wurde erzeugt "

#~ msgid "Initializing FUSE FILE ACCESS MANAGER !"
#~ msgstr "Initialisiere FUSE FILE ACCESS MANAGER!"

#~ msgid "Problem with '%(file)s' : %(error)s "
#~ msgstr "Problem mit '%(file)s' : %(error)s "

#~ msgid "Log settings"
#~ msgstr "Log-Einstellungen"

#~ msgid ""
#~ "Debug is just for debbuging purpose\n"
#~ "Info is the recommended setting"
#~ msgstr ""
#~ "Die empfohlene Einstellung ist 'Info'. Die Einstellung\n"
#~ "'Debug' ist für Entwicklerzwecke vorgesehen."

#~ msgid "Parameters"
#~ msgstr "Parameter"

#~ msgid "Security"
#~ msgstr "Sicherheit"

#~ msgid ""
#~ "'%(file)s' size is higher than the specified one ( %(filesize)s > %"
#~ "(maxsize)s), it has to be exclude "
#~ msgstr ""
#~ "'%(file)s' Dateigröße ist größer als festgelegte Maximalgröße ( %"
#~ "(filesize)s > %(maxsize)s), wird ausgelassen "

#~ msgid "got an error with '%(file)s' : %(error)s"
#~ msgstr "Fehler bei '%(file)s' : %(error)s"

#~ msgid "Target '%s' directory is excluded"
#~ msgstr "Zielverzeichnis '%s' wird ausgelassen"

#~ msgid "'%(file)s' doesn't exist, it has to be exclude "
#~ msgstr "'%(file)s' existiert nicht, wird ausgelassen "

#~ msgid ""
#~ "We don't have read access to '%(file)s', it has to be exclude : %(error)s "
#~ msgstr ""
#~ "Kein Lesezugriff auf '%(file)s' möglich, wird ausgelassen: %(error)s "

#~ msgid "Terminating FUSE FILE ACCESS MANAGER !"
#~ msgstr "Beenden des FUSE FILE ACCESS MANAGER !"

#~ msgid ""
#~ "When setting a username (resp password), password (resp username) is "
#~ "mandatory"
#~ msgstr ""
#~ "Beim Festlegen eines Benutzernamens (bzw. Kennwortes) ist die Angabe "
#~ "eines Kennwortes (bzw. Benutzernamens) zwingend"

#~ msgid "Choose the SSL option (smtptls=1) to be able to use Cert and Key"
#~ msgstr ""
#~ "Wählen Sie die Option SSL (smtptls=1) um ein Zertifikat und einen "
#~ "Schlüssel zu verwenden (Cert & Key)"

#~ msgid "I didn't find the log file. Please set it up in sbackup.conf "
#~ msgstr ""
#~ "Die Logdatei wurde nicht gefunden. Bitte legen Sie eine Logdatei in "
#~ "'sbackup.conf' fest. "

#~ msgid "No directory to backup !"
#~ msgstr "Kein Verzeichnis zu sichern!"

#~ msgid "Save to file ..."
#~ msgstr "Konfigurationsdatei speichern unter ..."

#~ msgid "Set the reciever of this mail"
#~ msgstr "Empfänger dieser E-Mail setzen"

#~ msgid ""
#~ "Please enter the new profile name in the <b>Profile name</b> entry and "
#~ "check the <b>Enable</b> checkbox to enable it."
#~ msgstr ""
#~ "Bitte geben Sie einen Namen für das neue Profil im Feld <b>Name des "
#~ "Profils</b> ein und setzen Sie einen Haken bei <b>Profil verwenden</b> um "
#~ "das Profil zu aktivieren."

#~ msgid "Got an exception when Pulling '%s' : "
#~ msgstr "Ausnahmefehler beim Zugriff auf Sicherung '%s': "

#~ msgid "Snapshot '%s' is a full . Can't rebase on older snapshot"
#~ msgstr ""
#~ "'%s' ist eine vollständige Sicherung. Datengrundlage kann nicht auf eine "
#~ "ältere Sicherung umgestellt werden."

#~ msgid "Deleting '%(snp)s' for purge !"
#~ msgstr "Aufräumen: '%(snp)s' wird gelöscht!"

#~ msgid "Non existing directory : %s"
#~ msgstr "Verzeichnis existiert nicht: %s"

#~ msgid "gtk-cancel"
#~ msgstr "gtk-cancel"

#~ msgid "gtk-no"
#~ msgstr "gtk-no"

#~ msgid "gtk-yes"
#~ msgstr "gtk-yes"

#~ msgid "gtk-close"
#~ msgstr "gtk-close"

#~ msgid "gtk-remove"
#~ msgstr "gtk-remove"

#~ msgid "gtk-add"
#~ msgstr "gtk-add"

#~ msgid "gtk-edit"
#~ msgstr "gtk-edit"

#~ msgid "gtk-ok"
#~ msgstr "gtk-ok"

#~ msgid "Test all"
#~ msgstr "Alle Testen"

#~ msgid "Starting snapshot %(name)s "
#~ msgstr "Beginne mit Erstellung der Datensicherung %(name)s "

#~ msgid "Time (for admins only)"
#~ msgstr "Automatische Sicherungen"

#~ msgid "Unable to change permissions for "
#~ msgstr "Zugriffsrechte konnten nicht geändert werden für "

#~ msgid "sbackup configuration error"
#~ msgstr "sbackup Konfigurationsfehler"

#~ msgid "This is a user friendly backup solution for common desktop needs."
#~ msgstr ""
#~ "Dies ist eine benutzerfreundliche Backup-Lösung für alltägliche Aufgaben "
#~ "im Desktop-Bereich."

#~ msgid "Terminating FUSE FILE ACCESS MANAGER!"
#~ msgstr "Beenden des FUSE File Access Manager!"

#~ msgid "File list ready, committing to disk."
#~ msgstr "Erstellen der Datei-Liste ist beendet, wird auf Platte geschrieben."

#~ msgid "No path to log file specified. Please set it in the configuration."
#~ msgstr ""
#~ "Kein Pfad zur Logdatei angegeben. Bitte einen Pfad in der Konfiguration "
#~ "festlegen."
